
<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
        <meta name="description" content="This web site provides an interface for NMR Metabolomics Protocols">
      
      
        <meta name="author" content="Rudi Grosman - University of Liverpool NMRF">
      
      
        <link rel="canonical" href="https://rgmetab.github.io/NMRmetab/print_page/">
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.4.0, mkdocs-material-8.5.6">
    
    
      
        <title>Print Site - NMR Metabolomics</title>
      
    
    
      <link rel="stylesheet" href="../assets/stylesheets/main.20d9efc8.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.cbb835fc.min.css">
        
      
      

    
    
    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../css/print-site-enum-headings1.css">
    
      <link rel="stylesheet" href="../css/print-site-enum-headings2.css">
    
      <link rel="stylesheet" href="../css/print-site-enum-headings3.css">
    
      <link rel="stylesheet" href="../css/print-site-enum-headings4.css">
    
      <link rel="stylesheet" href="../css/print-site-enum-headings5.css">
    
      <link rel="stylesheet" href="../css/print-site-enum-headings6.css">
    
      <link rel="stylesheet" href="../css/print-site.css">
    
      <link rel="stylesheet" href="../css/print-site-material.css">
    
    <script>__md_scope=new URL("/NMRmetab/",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  
        <script type="text/javascript">
        document.addEventListener('DOMContentLoaded', function () {
            remove_material_navigation();remove_mkdocs_theme_navigation();generate_toc();
        })
        </script>
        </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="" data-md-color-accent="">
  
    
    
      <script>var palette=__md_get("__palette");if(palette&&"object"==typeof palette.color)for(var key of Object.keys(palette.color))document.body.setAttribute("data-md-color-"+key,palette.color[key])</script>
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
      <div data-md-component="outdated" hidden>
        
      </div>
    
    
      

  

<header class="md-header md-header--lifted" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="NMR Metabolomics" class="md-header__button md-logo" aria-label="NMR Metabolomics" data-md-component="logo">
      
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54Z"/></svg>

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            NMR Metabolomics
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Print Site
            
          </span>
        </div>
      </div>
    </div>
    
      <form class="md-header__option" data-md-component="palette">
        
          
          
          <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="" data-md-color-accent=""  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_1">
          
            <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_2" hidden>
              <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
            </label>
          
        
          
          
          <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="" data-md-color-accent=""  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_2">
          
            <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_1" hidden>
              <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
            </label>
          
        
      </form>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
  </nav>
  
    
      
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-tabs__inner md-grid">
    <ul class="md-tabs__list">
      
        
  
  


  <li class="md-tabs__item">
    <a href=".." class="md-tabs__link">
      Welcome to the NMR Metabolomics protocol portal!
    </a>
  </li>

      
        
  
  


  
  
  
    

  
  
  
    

  
  
  
    <li class="md-tabs__item">
      <a href="../Data%20Analysis/Data%20Annotation/Chenomx%20Profiler/" class="md-tabs__link">
        Data Analysis
      </a>
    </li>
  

  

  

      
        
  
  


  
  
  
    

  
  
  
    <li class="md-tabs__item">
      <a href="../Miscellaneous/Tube%20washing/" class="md-tabs__link">
        Miscellaneous
      </a>
    </li>
  

  

      
        
  
  


  
  
  
    

  
  
  
    <li class="md-tabs__item">
      <a href="../Resources/Background%20Reading/" class="md-tabs__link">
        Resources
      </a>
    </li>
  

  

      
        
  
  


  
  
  
    

  
  
  
    <li class="md-tabs__item">
      <a href="../Sample%20Preparation/Blood/" class="md-tabs__link">
        Sample Preparation
      </a>
    </li>
  

  

      
    </ul>
  </div>
</nav>
    
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="NMR Metabolomics" class="md-nav__button md-logo" aria-label="NMR Metabolomics" data-md-component="logo">
      
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54Z"/></svg>

    </a>
    NMR Metabolomics
  </label>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
      

  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        Welcome to the NMR Metabolomics protocol portal!
      </a>
    </li>
  

    
      
      
      

  
  
  
    
    <li class="md-nav__item md-nav__item--nested">
      
      
        <input class="md-nav__toggle md-toggle" data-md-toggle="__nav_2" type="checkbox" id="__nav_2" >
      
      
      
        
          
        
          
        
          
        
      
      
        <label class="md-nav__link" for="__nav_2">
          Data Analysis
          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <nav class="md-nav" aria-label="Data Analysis" data-md-level="1">
        <label class="md-nav__title" for="__nav_2">
          <span class="md-nav__icon md-icon"></span>
          Data Analysis
        </label>
        <ul class="md-nav__list" data-md-scrollfix>
          
            
              
  
  
  
    
    <li class="md-nav__item md-nav__item--nested">
      
      
        <input class="md-nav__toggle md-toggle" data-md-toggle="__nav_2_1" type="checkbox" id="__nav_2_1" >
      
      
      
        
          
        
          
        
          
        
      
      
        <label class="md-nav__link" for="__nav_2_1">
          Data Annotation
          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <nav class="md-nav" aria-label="Data Annotation" data-md-level="2">
        <label class="md-nav__title" for="__nav_2_1">
          <span class="md-nav__icon md-icon"></span>
          Data Annotation
        </label>
        <ul class="md-nav__list" data-md-scrollfix>
          
            
              
  
  
  
    
    <li class="md-nav__item md-nav__item--nested">
      
      
        <input class="md-nav__toggle md-toggle" data-md-toggle="__nav_2_1_1" type="checkbox" id="__nav_2_1_1" >
      
      
      
        
          
            
          
        
      
      
        
        
        <div class="md-nav__link md-nav__link--index ">
          <a href="../Data%20Analysis/Data%20Annotation/Chenomx%20Profiler/">Chenomx Profiler</a>
          
        </div>
      
      <nav class="md-nav" aria-label="Chenomx Profiler" data-md-level="3">
        <label class="md-nav__title" for="__nav_2_1_1">
          <span class="md-nav__icon md-icon"></span>
          Chenomx Profiler
        </label>
        <ul class="md-nav__list" data-md-scrollfix>
          
            
          
        </ul>
      </nav>
    </li>
  

            
          
            
              
  
  
  
    
    <li class="md-nav__item md-nav__item--nested">
      
      
        <input class="md-nav__toggle md-toggle" data-md-toggle="__nav_2_1_2" type="checkbox" id="__nav_2_1_2" >
      
      
      
        
          
            
          
        
      
      
        
        
        <div class="md-nav__link md-nav__link--index ">
          <a href="../Data%20Analysis/Data%20Annotation/Correct%20annotation%20of%20Spreadsheet%20for%20import%20into%20IconNMR/">Correct annotation of Spreadsheet for import into IconNMR</a>
          
        </div>
      
      <nav class="md-nav" aria-label="Correct annotation of Spreadsheet for import into IconNMR" data-md-level="3">
        <label class="md-nav__title" for="__nav_2_1_2">
          <span class="md-nav__icon md-icon"></span>
          Correct annotation of Spreadsheet for import into IconNMR
        </label>
        <ul class="md-nav__list" data-md-scrollfix>
          
            
          
        </ul>
      </nav>
    </li>
  

            
          
            
              
  
  
  
    
    <li class="md-nav__item md-nav__item--nested">
      
      
        <input class="md-nav__toggle md-toggle" data-md-toggle="__nav_2_1_3" type="checkbox" id="__nav_2_1_3" >
      
      
      
        
          
            
          
        
      
      
        
        
        <div class="md-nav__link md-nav__link--index ">
          <a href="../Data%20Analysis/Data%20Annotation/Patter%20File%20Preparation/">Patter File Preparation</a>
          
        </div>
      
      <nav class="md-nav" aria-label="Patter File Preparation" data-md-level="3">
        <label class="md-nav__title" for="__nav_2_1_3">
          <span class="md-nav__icon md-icon"></span>
          Patter File Preparation
        </label>
        <ul class="md-nav__list" data-md-scrollfix>
          
            
          
        </ul>
      </nav>
    </li>
  

            
          
        </ul>
      </nav>
    </li>
  

            
          
            
              
  
  
  
    
    <li class="md-nav__item md-nav__item--nested">
      
      
        <input class="md-nav__toggle md-toggle" data-md-toggle="__nav_2_2" type="checkbox" id="__nav_2_2" >
      
      
      
        
          
        
          
        
      
      
        <label class="md-nav__link" for="__nav_2_2">
          Data Quality Control
          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <nav class="md-nav" aria-label="Data Quality Control" data-md-level="2">
        <label class="md-nav__title" for="__nav_2_2">
          <span class="md-nav__icon md-icon"></span>
          Data Quality Control
        </label>
        <ul class="md-nav__list" data-md-scrollfix>
          
            
              
  
  
  
    
    <li class="md-nav__item md-nav__item--nested">
      
      
        <input class="md-nav__toggle md-toggle" data-md-toggle="__nav_2_2_1" type="checkbox" id="__nav_2_2_1" >
      
      
      
        
          
            
          
        
      
      
        
        
        <div class="md-nav__link md-nav__link--index ">
          <a href="../Data%20Analysis/Data%20Quality%20Control/TameNMR/">TameNMR</a>
          
        </div>
      
      <nav class="md-nav" aria-label="TameNMR" data-md-level="3">
        <label class="md-nav__title" for="__nav_2_2_1">
          <span class="md-nav__icon md-icon"></span>
          TameNMR
        </label>
        <ul class="md-nav__list" data-md-scrollfix>
          
            
          
        </ul>
      </nav>
    </li>
  

            
          
            
              
  
  
  
    
    <li class="md-nav__item md-nav__item--nested">
      
      
        <input class="md-nav__toggle md-toggle" data-md-toggle="__nav_2_2_2" type="checkbox" id="__nav_2_2_2" >
      
      
      
        
          
            
          
        
      
      
        
        
        <div class="md-nav__link md-nav__link--index ">
          <a href="../Data%20Analysis/Data%20Quality%20Control/Topspin%20Quality%20Control/">Topspin Quality Control</a>
          
        </div>
      
      <nav class="md-nav" aria-label="Topspin Quality Control" data-md-level="3">
        <label class="md-nav__title" for="__nav_2_2_2">
          <span class="md-nav__icon md-icon"></span>
          Topspin Quality Control
        </label>
        <ul class="md-nav__list" data-md-scrollfix>
          
            
          
        </ul>
      </nav>
    </li>
  

            
          
        </ul>
      </nav>
    </li>
  

            
          
            
              
  
  
  
    
    <li class="md-nav__item md-nav__item--nested">
      
      
        <input class="md-nav__toggle md-toggle" data-md-toggle="__nav_2_3" type="checkbox" id="__nav_2_3" >
      
      
      
        
          
        
          
        
      
      
        <label class="md-nav__link" for="__nav_2_3">
          Statistical Analyses
          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <nav class="md-nav" aria-label="Statistical Analyses" data-md-level="2">
        <label class="md-nav__title" for="__nav_2_3">
          <span class="md-nav__icon md-icon"></span>
          Statistical Analyses
        </label>
        <ul class="md-nav__list" data-md-scrollfix>
          
            
              
  
  
  
    
    <li class="md-nav__item md-nav__item--nested">
      
      
        <input class="md-nav__toggle md-toggle" data-md-toggle="__nav_2_3_1" type="checkbox" id="__nav_2_3_1" >
      
      
      
        
          
            
          
        
      
      
        
        
        <div class="md-nav__link md-nav__link--index ">
          <a href="../Data%20Analysis/Statistical%20Analyses/Pathway%20Analysis%20in%20MetaboAnalyst/">Pathway Analysis in MetaboAnalyst</a>
          
        </div>
      
      <nav class="md-nav" aria-label="Pathway Analysis in MetaboAnalyst" data-md-level="3">
        <label class="md-nav__title" for="__nav_2_3_1">
          <span class="md-nav__icon md-icon"></span>
          Pathway Analysis in MetaboAnalyst
        </label>
        <ul class="md-nav__list" data-md-scrollfix>
          
            
          
        </ul>
      </nav>
    </li>
  

            
          
            
              
  
  
  
    
    <li class="md-nav__item md-nav__item--nested">
      
      
        <input class="md-nav__toggle md-toggle" data-md-toggle="__nav_2_3_2" type="checkbox" id="__nav_2_3_2" >
      
      
      
        
          
            
          
        
      
      
        
        
        <div class="md-nav__link md-nav__link--index ">
          <a href="../Data%20Analysis/Statistical%20Analyses/Statistical%20Analysis%20in%20MetaboAnalyst/">Statistical Analysis in MetaboAnalyst</a>
          
        </div>
      
      <nav class="md-nav" aria-label="Statistical Analysis in MetaboAnalyst" data-md-level="3">
        <label class="md-nav__title" for="__nav_2_3_2">
          <span class="md-nav__icon md-icon"></span>
          Statistical Analysis in MetaboAnalyst
        </label>
        <ul class="md-nav__list" data-md-scrollfix>
          
            
          
        </ul>
      </nav>
    </li>
  

            
          
        </ul>
      </nav>
    </li>
  

            
          
        </ul>
      </nav>
    </li>
  

    
      
      
      

  
  
  
    
    <li class="md-nav__item md-nav__item--nested">
      
      
        <input class="md-nav__toggle md-toggle" data-md-toggle="__nav_3" type="checkbox" id="__nav_3" >
      
      
      
        
          
        
      
      
        <label class="md-nav__link" for="__nav_3">
          Miscellaneous
          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <nav class="md-nav" aria-label="Miscellaneous" data-md-level="1">
        <label class="md-nav__title" for="__nav_3">
          <span class="md-nav__icon md-icon"></span>
          Miscellaneous
        </label>
        <ul class="md-nav__list" data-md-scrollfix>
          
            
              
  
  
  
    
    <li class="md-nav__item md-nav__item--nested">
      
      
        <input class="md-nav__toggle md-toggle" data-md-toggle="__nav_3_1" type="checkbox" id="__nav_3_1" >
      
      
      
        
          
            
          
        
      
      
        
        
        <div class="md-nav__link md-nav__link--index ">
          <a href="../Miscellaneous/Tube%20washing/">Tube washing</a>
          
        </div>
      
      <nav class="md-nav" aria-label="Tube washing" data-md-level="2">
        <label class="md-nav__title" for="__nav_3_1">
          <span class="md-nav__icon md-icon"></span>
          Tube washing
        </label>
        <ul class="md-nav__list" data-md-scrollfix>
          
            
          
        </ul>
      </nav>
    </li>
  

            
          
        </ul>
      </nav>
    </li>
  

    
      
      
      

  
  
  
    
    <li class="md-nav__item md-nav__item--nested">
      
      
        <input class="md-nav__toggle md-toggle" data-md-toggle="__nav_4" type="checkbox" id="__nav_4" >
      
      
      
        
          
        
      
      
        <label class="md-nav__link" for="__nav_4">
          Resources
          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <nav class="md-nav" aria-label="Resources" data-md-level="1">
        <label class="md-nav__title" for="__nav_4">
          <span class="md-nav__icon md-icon"></span>
          Resources
        </label>
        <ul class="md-nav__list" data-md-scrollfix>
          
            
              
  
  
  
    
    <li class="md-nav__item md-nav__item--nested">
      
      
        <input class="md-nav__toggle md-toggle" data-md-toggle="__nav_4_1" type="checkbox" id="__nav_4_1" >
      
      
      
        
          
            
          
        
      
      
        
        
        <div class="md-nav__link md-nav__link--index ">
          <a href="../Resources/Background%20Reading/">Background Reading</a>
          
        </div>
      
      <nav class="md-nav" aria-label="Background Reading" data-md-level="2">
        <label class="md-nav__title" for="__nav_4_1">
          <span class="md-nav__icon md-icon"></span>
          Background Reading
        </label>
        <ul class="md-nav__list" data-md-scrollfix>
          
            
          
        </ul>
      </nav>
    </li>
  

            
          
        </ul>
      </nav>
    </li>
  

    
      
      
      

  
  
  
    
    <li class="md-nav__item md-nav__item--nested">
      
      
        <input class="md-nav__toggle md-toggle" data-md-toggle="__nav_5" type="checkbox" id="__nav_5" >
      
      
      
        
          
        
          
        
          
        
          
        
          
        
      
      
        <label class="md-nav__link" for="__nav_5">
          Sample Preparation
          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <nav class="md-nav" aria-label="Sample Preparation" data-md-level="1">
        <label class="md-nav__title" for="__nav_5">
          <span class="md-nav__icon md-icon"></span>
          Sample Preparation
        </label>
        <ul class="md-nav__list" data-md-scrollfix>
          
            
              
  
  
  
    
    <li class="md-nav__item md-nav__item--nested">
      
      
        <input class="md-nav__toggle md-toggle" data-md-toggle="__nav_5_1" type="checkbox" id="__nav_5_1" >
      
      
      
        
          
            
          
        
      
      
        
        
        <div class="md-nav__link md-nav__link--index ">
          <a href="../Sample%20Preparation/Blood/">Blood</a>
          
        </div>
      
      <nav class="md-nav" aria-label="Blood" data-md-level="2">
        <label class="md-nav__title" for="__nav_5_1">
          <span class="md-nav__icon md-icon"></span>
          Blood
        </label>
        <ul class="md-nav__list" data-md-scrollfix>
          
            
          
        </ul>
      </nav>
    </li>
  

            
          
            
              
  
  
  
    
    <li class="md-nav__item md-nav__item--nested">
      
      
        <input class="md-nav__toggle md-toggle" data-md-toggle="__nav_5_2" type="checkbox" id="__nav_5_2" >
      
      
      
        
          
            
          
        
      
      
        
        
        <div class="md-nav__link md-nav__link--index ">
          <a href="../Sample%20Preparation/Cells/">Cells</a>
          
        </div>
      
      <nav class="md-nav" aria-label="Cells" data-md-level="2">
        <label class="md-nav__title" for="__nav_5_2">
          <span class="md-nav__icon md-icon"></span>
          Cells
        </label>
        <ul class="md-nav__list" data-md-scrollfix>
          
            
          
        </ul>
      </nav>
    </li>
  

            
          
            
              
  
  
  
    
    <li class="md-nav__item md-nav__item--nested">
      
      
        <input class="md-nav__toggle md-toggle" data-md-toggle="__nav_5_3" type="checkbox" id="__nav_5_3" >
      
      
      
        
          
            
          
        
      
      
        
        
        <div class="md-nav__link md-nav__link--index ">
          <a href="../Sample%20Preparation/Cells%20Intracellular/">Cells Intracellular</a>
          
        </div>
      
      <nav class="md-nav" aria-label="Cells Intracellular" data-md-level="2">
        <label class="md-nav__title" for="__nav_5_3">
          <span class="md-nav__icon md-icon"></span>
          Cells Intracellular
        </label>
        <ul class="md-nav__list" data-md-scrollfix>
          
            
          
        </ul>
      </nav>
    </li>
  

            
          
            
              
  
  
  
    
    <li class="md-nav__item md-nav__item--nested">
      
      
        <input class="md-nav__toggle md-toggle" data-md-toggle="__nav_5_4" type="checkbox" id="__nav_5_4" >
      
      
      
        
          
            
          
        
      
      
        
        
        <div class="md-nav__link md-nav__link--index ">
          <a href="../Sample%20Preparation/Tissues/">Tissues</a>
          
        </div>
      
      <nav class="md-nav" aria-label="Tissues" data-md-level="2">
        <label class="md-nav__title" for="__nav_5_4">
          <span class="md-nav__icon md-icon"></span>
          Tissues
        </label>
        <ul class="md-nav__list" data-md-scrollfix>
          
            
          
        </ul>
      </nav>
    </li>
  

            
          
            
              
  
  
  
    
    <li class="md-nav__item md-nav__item--nested">
      
      
        <input class="md-nav__toggle md-toggle" data-md-toggle="__nav_5_5" type="checkbox" id="__nav_5_5" >
      
      
      
        
          
            
          
        
      
      
        
        
        <div class="md-nav__link md-nav__link--index ">
          <a href="../Sample%20Preparation/Urine/">Urine</a>
          
        </div>
      
      <nav class="md-nav" aria-label="Urine" data-md-level="2">
        <label class="md-nav__title" for="__nav_5_5">
          <span class="md-nav__icon md-icon"></span>
          Urine
        </label>
        <ul class="md-nav__list" data-md-scrollfix>
          
            
          
        </ul>
      </nav>
    </li>
  

            
          
        </ul>
      </nav>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#" class="md-nav__link">
    1. Welcome to the NMR Metabolomics protocol portal!
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#section-data-analysis" class="md-nav__link">
    I. Data Analysis
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#section-miscellaneous" class="md-nav__link">
    II. Miscellaneous
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#section-resources" class="md-nav__link">
    III. Resources
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#section-sample-preparation" class="md-nav__link">
    IV. Sample Preparation
  </a>
  
</li>
      
    </ul>
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  


<div id="print-site-page" class="print-site-enumerate-headings print-site-enumerate-figures">
        <section class="print-page">
            <div id="print-page-toc" data-toc-depth="3">
                <nav role='navigation' class='print-page-toc-nav'>
                <h1 class='print-page-toc-title'>Table of Contents</h1>
                </nav>
            </div>
        </section>
        <section class="print-page" id="index"><h1 id="index-welcome-to-the-nmr-metabolomics-protocol-portal">Welcome to the NMR Metabolomics protocol portal!</h1>
<h2 id="index-contents">Contents</h2>
<p>Here you can find the protocols used by the NMR Metabolomics group. Protocols are collated in following groups:</p>
<ol>
<li>Sample Preparation</li>
<li>Data Acquisition</li>
<li>Data Analysis</li>
</ol>
<h2 id="index-usage">Usage</h2>
<p>Individual protocols can be selected by expanding the categories on the left hand side.
Protocols or contents can be searched through the magnifying glass icon on the top right.</p>
<h2 id="index-adding-modifying-or-reporting">Adding, Modifying or Reporting.</h2>
<p>Please contact <a href="mailto:r.grosman@liverpool.ac.uk" title="r.grosman@liverpool.ac.uk">Rudi Grosman</a> for addition modification of protocols as well as reporting bugs.</p>
<p>Good luck with your experiments and remember:</p>
<blockquote>
<p>If at first you don't succeed try two more times so that your failure is statistically significant!</p>
</blockquote></section>
                        <h1 class='nav-section-title' id='section-data-analysis'>
                            Data Analysis <a class='headerlink' href='#section-data-analysis' title='Permanent link'>↵</a>
                        </h1>
                        
                        <h2 class='nav-section-title' id='section-data-annotation'>
                            Data Annotation <a class='headerlink' href='#section-data-annotation' title='Permanent link'>↵</a>
                        </h2>
                        
                        <h3 class='nav-section-title' id='section-chenomx-profiler'>
                            Chenomx Profiler <a class='headerlink' href='#section-chenomx-profiler' title='Permanent link'>↵</a>
                        </h3>
                        <section class="print-page" id="data%20analysis-data%20annotation-chenomx%20profiler"><h1 id="data%20analysis-data%20annotation-chenomx%20profiler-chenomx-profiler-metabolite-identification-quick-guide">Chenomx Profiler Metabolite Identification - Quick Guide</h1>
<p>Author: Rudi Grosman &amp; Marie Phenlan
Revision Date: 2 November 2017</p>
<p>Chenomx is a commercial software package for which we have two 'standard' licenses and one 'professional' license. Software may also be used as 'evaluation' which allows metabolite identification but you cannot save your results. All licensed versions enable saving and the professional version allows import and build of new molecules for the library. Book the appropriate hot-desk as required:</p>
<ul>
<li>METAB4 Chenomx Professional </li>
<li>METAB1 Chenomx Standard </li>
<li>METAB2 Chenomx Standard</li>
</ul>
<p>Chenomx works ideally with spectra acquired at 25&deg;C and around 7.0 pH.</p>
<p>1) <strong>Launch Chenomx</strong> </p>
<p>Launch Chenomx from the desktop icon or the panel shortcut (select profiler if prompted):</p>
<p>When launched Chenomx will ask for a licenses (on METAB3, TD, LNX7 and LNX6).</p>
<p>Click on <strong>Evaluate</strong>.</p>
<p>The software launches with a message offering the option to analyse a sample spectrum. Click on ‘No’. </p>
<p>2) <strong>Opening a spectrum in Chenomx</strong> </p>
<p>On the menubar click on File and select Open to proceed to the open dialogue. NMR spectra (Bruker format) are saved in a file called “1r” which are typically located in a nested file directory structure (<span style="color:fuchsia">replace pink text with relevant information</span>): /home/<span style="color:fuchsia">[USER NAME]</span>/data/<span style="color:fuchsia">[DATA NAME]</span>/nmr/<span style="color:fuchsia">[EXPERIMENT NUMBER]</span>/pdata/1/1r and click ‘Open’. </p>
<p>3) <strong>Calibrate &amp; Process spectra</strong></p>
<p>An import screen will request sample details in order to calibrate the built-in metabolite library for analysis.</p>
<p>​    <strong>Calibration:</strong></p>
<p>​        In most biofluids (not blood, SF or CSF) samples will have an internal standard (Chemical Shape Indicator (CSI) in Chenomx). The most common standard is TSP, but this may be different in certain cases. Refer to the sample preparation documents if unsure.</p>
<p>​        Once selected enter the final concentration of the standard TSP in the sample. This information will be used for identification and quantification of metabolites.</p>
<p>​        Uncheck the “Sample contains one or more pH Indicators” box and enter the sample pH value (7.4) followed by +/- value for adjustment. Metabolites will have shifted signals at different pHs - this information will calibrate the shifts of metabolites for analysis.</p>
<p>​    <strong>Processing:</strong></p>
<p>​        In metabolomics the spectra is automatically processed immediately after acquisition therefore no extra processing <em>should</em> be required (leave this unchecked).</p>
<p>Click <strong>OK</strong>.</p>
<p>Chenomx will now load the selected spectrum for analysis.</p>
<p>4) <strong>Spectrum Manipulation</strong></p>
<p>​    After the spectrum is loaded the analysis screen opens. Spectra can be scaled (y axis) up and down via scrolling mouse wheel down and up (respectively). Zoom in / out (x axis) via pressing shift + scrolling mouse wheel. </p>
<p>5) <strong>Tools for analysis</strong></p>
<p>The left hand panel provides 5 tabs to aid with analysis:</p>
<p><strong>Reference Card:</strong> Reference cards give information and links on selected metabolites.</p>
<p><strong>Compound Sets:</strong> Chenomx allows the use of in-house metabolite libraries, from this tab an in-house library may be selected (if available).</p>
<p><strong>Spectrum Details:</strong> Displays details of current spectrum (instrument, field strength, experiments, etc.).</p>
<p><strong>Legend:</strong> Provides control to work on layers of spectra, as well as to create digital spectrum via summing or subtracting spectrum digitally.</p>
<p><strong>Files:</strong> File browser to swap between spectrum quickly.</p>
<p>6) <strong>Manual Metabolite analysis</strong></p>
<p>To start analysis, select a metabolite from the compound list. When selected, the compound spectrum will be overlaid in blue on your sample spectrum.</p>
<p>Dashed blue lines shows the shifts of the peaks for the selected metabolite. On the upper left corner right under the compound name. All peaks of the compound are listed and when clicked it will automatically zoom in to that region allowing finer adjustments for your analysis.</p>
<p>7) <strong>Automated Metabolite Analysis</strong></p>
<p>What makes Chenomx a faster streamlined metabolite identifier is the feature that automatically fits multiple compounds via recreating your spectrum with the built-in library through a series of algorithms (this is why upon importing it is very important to provide the correct standard (as well as its concentration) and pH value).</p>
<p>Select the entire range of metabolites listed below the spectrum panel by clicking on the panel and typing ctrl+A . Alternatively select a subset of metabolites using the mouse and shift key (individual metabolites can be added and taken out via control+click).</p>
<p>Right click on your selection and click on “Fit Automatically”.</p>
<p>After Chenomx finishes compound fitting, depending on the metabolites selected you will encounter two info messages.</p>
<p>If metabolites were matched, Click <strong>OK</strong> and proceed to manual verification.</p>
<p>If no compounds were matched click <strong>OK</strong> and either try another set of compounds or a different spectrum (the import options may require adjustment). </p>
<p>8) <strong>Automated Metabolite Analysis Appraisal</strong></p>
<p>Matched compounds will now be shown in bold letters in the compound list and they will also be quantified. At this point metabolite fits should be appraised.</p>
<p>The <span style="color:black">black </span> spectrum is your original spectrum.</p>
<p>The <span style="color:red">red </span>spectrum is a digitally created spectrum from the fitted compounds.</p>
<p><span style="color:blue">Blue </span> arrows shows the peaks of the currently selected compound (you can switch to reference card tabs to obtain more information about your compounds).</p>
<p>On the upper left corner just below the compound name in the spectrum window there is only one value “2.4” which show the corresponding peak of the compound. This value can appear in two conditions:</p>
<ul>
<li>Green: The peak is fitted to the original spectrum.</li>
<li>Black: The peak might fit but requires fine tuning.</li>
</ul>
<p>NMR provides unique characteristic spectrum for each molecule although their location on a spectrum might change slightly depending on conditions (i.e temperature, pH, other compounds in the solution) the overall fingerprint is always the same. This is why when Chenomx flags a peak that is not green it you can adjust the intensity and shift within a certain margin of error. Click the peak in question then hold down left mouse button over the blue shaded area and drag the peak to change height and or shift.</p>
<blockquote>
<p>For a molecule with multiple peaks the relative height volume is fixed therefore be mindful when adjusting one signal that <strong>all</strong> signals corresponding to that molecule will change.</p>
</blockquote>
<p>9) <strong>Peak Matching</strong></p>
<p>When a compound is fitted but peaks are not matching, click on the peaks that are in black below the compound name.</p>
<p>This will zoom in to that peak region where you can move the peak around with in the given margin.</p>
<p>When you are out of range your peak indicator will turn red.</p>
<p>When you fit the peak it will turn green.</p>
<p>Some peaks may not turn green because they will under the other peaks or noise, that is why you need to verify each match given by chenomx.</p>
<p>When you are done with your verification, select another metabolite and you will be seeing its projection on your spectrum. Here you can clearly see that NADP+ is in the spectrum but all peaks appear to be black.</p>
<p>After adjustment even though all the peaks are not green the fit is clear and the metabolite is identified. </p></section><h1 class='nav-section-title-end'>Ended: Chenomx Profiler</h1>
                        <h3 class='nav-section-title' id='section-correct-annotation-of-spreadsheet-for-import-into-iconnmr'>
                            Correct annotation of Spreadsheet for import into IconNMR <a class='headerlink' href='#section-correct-annotation-of-spreadsheet-for-import-into-iconnmr' title='Permanent link'>↵</a>
                        </h3>
                        <section class="print-page" id="data%20analysis-data%20annotation-correct%20annotation%20of%20spreadsheet%20for%20import%20into%20iconnmr"><h1 id="data%20analysis-data%20annotation-correct%20annotation%20of%20spreadsheet%20for%20import%20into%20iconnmr-correct-annotation-of-spreadsheet-for-import-into-iconnmr">Correct annotation of Spreadsheet for import into IconNMR</h1>
<p>Authors: Marie Phelan &amp; Rudi Grosman 
Revision Date: 1 November 2017</p>
<p>A spreadsheet should be provided with each batch of samples the information of this spreadsheet will be used to annotate the sample spectra therefore it is important to be as clear as possible with information.</p>
<p>Please fill columns out in the following order</p>
<p>A .csv file can be created using any text or spreadsheet editor.</p>
<p>Essential inputs are:</p>
<ul>
<li>
<p><strong>Study Name and Date [COL_A]:</strong> File name in which the results will be saved. Please do NOT use spaces or special characters - only use text, number &amp; underscore. For example Plasma_antibiotics_150114 is a plasma antibiotic study submitted on 15th October 2014.</p>
</li>
<li>
<p><strong>Solvent [COL_B]:</strong> Which solvent is used in your sample? These depend on biomaterial and extraction procedure; for biofluids (blood, urine etc) state “H2O+D2O”, aqueous cell or tissue extracts “D2O”, lipophilic extracts either “MeOD” or “CDCl3” – if in doubt ask</p>
</li>
<li>
<p><strong>Experiment [COL_C]:</strong> These are experiments set at the spectrometer – this can be left blank or set to the default is PROF_noesy </p>
</li>
<li>
<p><strong>Position in NMR rack [COL_D]:</strong> These are the positions of the samples as prepared for NMR – if not yet prepared for NMR leave this blank.</p>
</li>
<li>
<p><strong>Unique identifier [COL_E]:</strong> researchers own short-hand identity code for the sample.</p>
</li>
<li>
<p><strong>Storage Tube ID [COL_F]:</strong> The annotation on the storage tube also should be recorded -for clarity we recommend numbering consecutively from 1 to n (n = total number of samples). If samples in 96-well block provide row and column position. A1, B1, C1, D1, E1, F1, G1, H1 then A2, B2, C2, D2, E2, F2, G2, H2 etc.</p>
</li>
<li>
<p><strong>Cohort [COL_G] and replicate [COL_H]:</strong> identity for the cohort and biological/technical replicate number. If time course is provided this may be added as an additional column.</p>
</li>
<li>
<p><strong>Study details [COL_I]:</strong> provide details of the study (investigator, institute, area of study)</p>
</li>
</ul>
<p>Example of Spreadsheet format of a .csv file:</p>
<table>
<thead>
<tr>
<th>COL_A</th>
<th>COL_B</th>
<th>COL_C</th>
<th>COL_D</th>
<th>COL_E</th>
<th>COL_F</th>
<th>COL_G</th>
<th>COL_H</th>
<th>COL_I</th>
</tr>
</thead>
<tbody>
<tr>
<td>Plasma_antibiotics_150114</td>
<td>H2O+D2O</td>
<td>PROF_noesy</td>
<td>A1</td>
<td>PT121</td>
<td>121</td>
<td>Control</td>
<td>1</td>
<td>Clinical study Li-Hep Plasma</td>
</tr>
<tr>
<td>Plasma_antibiotics_150114</td>
<td>H2O+D2O</td>
<td>PROF_noesy</td>
<td>B1</td>
<td>PT124</td>
<td>124</td>
<td>Control</td>
<td>2</td>
<td>Clinical study Li-Hep Plasma</td>
</tr>
<tr>
<td>Plasma_antibiotics_150114</td>
<td>H2O+D2O</td>
<td>PROF_noesy</td>
<td>C1</td>
<td>PT127</td>
<td>127</td>
<td>Control</td>
<td>3</td>
<td>Clinical study Li-Hep Plasma</td>
</tr>
<tr>
<td>Plasma_antibiotics_150114</td>
<td>H2O+D2O</td>
<td>PROF_noesy</td>
<td>D1</td>
<td>PT131</td>
<td>131</td>
<td>Infection</td>
<td>1</td>
<td>Clinical study Li-Hep Plasma</td>
</tr>
<tr>
<td>Plasma_antibiotics_150114</td>
<td>H2O+D2O</td>
<td>PROF_noesy</td>
<td>E1</td>
<td>PT144</td>
<td>144</td>
<td>Infection</td>
<td>2</td>
<td>Clinical study Li-Hep Plasma</td>
</tr>
</tbody>
</table>
<p>PROF_noesy A1 PT121 121 Control 1 Clinical study Li-Hep Plasma Plasma_antibiotics_150114 H2O+D2O PROF_noesy B1 PT124 124 Control 2 Clinical study Li-Hep Plasma Plasma_antibiotics_150114 H2O+D2O PROF_noesy C1 PT127 127 Control 3 Clinical study Li-Hep Plasma Plasma_antibiotics_150114 H2O+D2O PROF_noesy D1 PT131 131 Infection 1 Clinical study Li-Hep Plasma Plasma_antibiotics_150114 H2O+D2O PROF_noesy E1 PT144 144 Infection 2 Clinical study Li-Hep Plasma</p>
<ul>
<li>All the information entered is case sensitive so make sure it is exactly the same.</li>
<li>When saving csv file ensure field delimiter is set to comma <code>,</code>.</li>
<li><strong>DO NOT</strong> use commas anywhere in the file Once done make sure you save the spreadsheet as a .csv file and check it with a text editor (such as notepad) to make sure it fits the format required. </li>
</ul></section><h1 class='nav-section-title-end'>Ended: Correct annotation of Spreadsheet for import into IconNMR</h1>
                        <h3 class='nav-section-title' id='section-patter-file-preparation'>
                            Patter File Preparation <a class='headerlink' href='#section-patter-file-preparation' title='Permanent link'>↵</a>
                        </h3>
                        <section class="print-page" id="data%20analysis-data%20annotation-patter%20file%20preparation"><h1 id="data%20analysis-data%20annotation-patter%20file%20preparation-pattern-file-preparation-guide">Pattern File Preparation Guide</h1>
<p>Authors: Rudi Grosman &amp; Marie Phelan
Revision Date: 1 November 2017</p>
<p>A pattern file can enhance the ease of analysis and identification of metabolites influencing statistical groupings. Pattern files form the basis for bucketing in AMIX enabling metabolite details to be encoded in the matrix of peak intensities taken forward for statistical analysis.</p>
<p><img alt="PatternTable" src="../Data%20Analysis/Data%20Annotation/Patter%20File%20Preparation/Pattern_Table.jpg" /></p>
<p>The format of the pattern file must be adhered to – any deviation may corrupt AMIX bucketing. Spaces and row information is critical for this Plain text document therefore it is recommended that the file is prepared using a text editor such as nedit, gedit or notepad in order to avoid undesired formatting.</p>
<h2 id="data%20analysis-data%20annotation-patter%20file%20preparation-header-section">Header Section</h2>
<p>The pattern file is made of two main sections; the header (lines 1-9, light grey) and the data (lines 10-20, dark grey).</p>
<p>The header must contain all the information required after the equal sign.</p>
<p>The ITEMS line must have the exact number of lines provided in the data section (lines 10-20) otherwise AMIX will accept the pattern file.</p>
<h2 id="data%20analysis-data%20annotation-patter%20file%20preparation-data-section">Data Section</h2>
<p>The data section contains left and right boundaries (in ppm) of the peaks and any annotations or identification for the peak.</p>
<p>First two columns must be left as 0.0000 and the following two columns should be the <strong>left</strong> and <strong>right chemical shift positions</strong> that define the peak.</p>
<blockquote>
<p>Shifts: Shifts (aka. Chemical shifts) are specific points on your spectrum which are
measured in ppm (parts per million). These value can be identified interactively using TopSpin.
Make sure that that you are reading this information in ppm and not in Hz (Hertz).</p>
</blockquote>
<p>Template pattern files are available on VOCAL to ensure correct formatting.</p>
<p>The line has a specific outline:
    Each * represents a space character:
<code>**0.0000***0.0000**4.2554****4.2461****0*glyceryl backbone CH2-2</code></p>
<p>The annotations must be unique, if not you won't be able to locate a specific peak during your
statistical analyses.</p>
<h2 id="data%20analysis-data%20annotation-patter%20file%20preparation-key-points">Key Points</h2>
<ul>
<li>Always use plain text.</li>
<li>Never use a word processor (e.g MS Word, OpenOffice, etc).</li>
<li>If prepared on Windows always check you file on linux text editor (e.g Nedit). Files prepared on Windows may introduce “<cr>” in your line. If this happens simply delete every <cr> and save the file before you proceed.</li>
</ul></section><h1 class='nav-section-title-end'>Ended: Patter File Preparation</h1><h1 class='nav-section-title-end'>Ended: Data Annotation</h1>
                        <h2 class='nav-section-title' id='section-data-quality-control'>
                            Data Quality Control <a class='headerlink' href='#section-data-quality-control' title='Permanent link'>↵</a>
                        </h2>
                        
                        <h3 class='nav-section-title' id='section-tamenmr'>
                            TameNMR <a class='headerlink' href='#section-tamenmr' title='Permanent link'>↵</a>
                        </h3>
                        <section class="print-page" id="data%20analysis-data%20quality%20control-tamenmr"><h1 id="data%20analysis-data%20quality%20control-tamenmr-tame-nmr">Tame NMR</h1>
<p>Author: Computational Biology Facility
Revision Date: July 2017</p>
<p>Developed by the University of Liverpool Computational Biology Facility as an open source tool to analyse metabolomics data (clearly and reproducibly).</p>
<p>Dr Arturas Grauslys (funded by A Jones - TRDF) has built a server-based analytical platform in the Galaxy framework that supports tools employed by MS metabolomics and proteomics as part of the COSMOS consortium.</p>
<p>The tool aims to streamline and pipe multiple NMR metabolomics analytical modules that can then be used to reproduce analyses and visualise the data as it is manipulated. An alpha test is available within the university of Liverpool with plans to open up the server to www in Autumn 2017.</p>
<p>One particular feature that has been useful for curration of datasets for deposition is the ability to visualise the pattern file (i.e. the bucket boundaries and their annotations) overtop the spectra. The following instructions describe how this can be done using TameNMR. </p>
<h2 id="data%20analysis-data%20quality%20control-tamenmr-prepare-nmr-spectra">Prepare NMR Spectra</h2>
<p>In order to overlay the pattern with the spectra the spectra must be converted into a recognisable format. We can use AMIX to export the spectra as a matrix of numbers but we need the narrowest bucket width possible (essentially ppm per point) to see true resolution. An alternative approach would be to export 1Ds as jcamp however this would need to be done for each spectrum individually (no way to process as a batch unlike AMIX).</p>
<ul>
<li>
<p>Open topspin and load dataset </p>
</li>
<li>
<p>Check number of points in processed spectrum </p>
</li>
</ul>
<blockquote>
<p>This can be found in Topspin under procPars tab SI e.g. 131072 </p>
</blockquote>
<ul>
<li>Check the HzpPT</li>
</ul>
<blockquote>
<p>Hertz per point – also under procPars tab e.g. 0.091699</p>
</blockquote>
<ul>
<li>Divide that by field strength to get ppm per point <blockquote>
<p>e.g. 0.091699/600.1299368=0.000153ppm per point</p>
</blockquote>
</li>
</ul>
<h2 id="data%20analysis-data%20quality%20control-tamenmr-export-spectra-as-csv-file">Export Spectra as csv file</h2>
<p>Follow general bucketing protocol for AMIX however set bucket width to per point value. </p>
<ul>
<li>Launch AMIX and carry out bucketing procedure with the following options </li>
<li>Simple rectangular buckets</li>
<li>No scaling</li>
<li>Sum of intensities</li>
<li>Set bucket widths to 0.000153 (or whatever value was calculated in 1.)</li>
<li>Export bucket table as csv with:</li>
<li>Tab separations</li>
<li>Spectra in columns</li>
</ul>
<h2 id="data%20analysis-data%20quality%20control-tamenmr-upload-csv-spectra-file-to-galaxy">Upload csv spectra file to galaxy</h2>
<p>Galaxy server can be accessed from anywhere within the University (or via VPN).</p>
<ul>
<li>Open galaxy in browser: <a href="../Data%20Analysis/Data%20Quality%20Control/TameNMR/galaxy.liv.ac.uk">galaxy.liv.ac.uk</a></li>
<li>Go to Get Data (Left hand) </li>
<li>Upload File (can drag and drop into upload window)</li>
<li>Leave type as Auto-detect and click Start</li>
<li>Wait for it to upload (this may take some time – worth carrying out analysis on SUBSET of spectra)</li>
</ul>
<h2 id="data%20analysis-data%20quality%20control-tamenmr-upload-patter-file">Upload Patter file</h2>
<h3 id="data%20analysis-data%20quality%20control-tamenmr-upload-the-bruker-format-pattern-file">Upload the Bruker format pattern file</h3>
<p>The pattern file is a Bruker formatted text file containing the bucket boundaries and their annotations for the dataset.</p>
<ul>
<li>Go to Get Data (Left hand) </li>
<li>Upload File (can drag and drop into upload window)</li>
<li>Leave type as Auto-detect and click Start</li>
<li>Now convert data with Import data tab</li>
<li>Prepare pattern file</li>
<li>Source – bruker pattern file</li>
<li>Bruker pattern file - use pattern file</li>
<li>Click execute</li>
</ul>
<h3 id="data%20analysis-data%20quality%20control-tamenmr-upload-the-bruker-format-pattern-file_1">Upload the bruker format pattern file</h3>
<p>The Bruker formatted text file containing the bucket boundaries and their annotations can be converted into a 3 column (tab separated) csv file and thus used directly in Galaxy without import/conversion. The 3 columns must be left bound (in ppm), right bound (in ppm) and annotation.</p>
<h2 id="data%20analysis-data%20quality%20control-tamenmr-plot-the-overlay">Plot the Overlay</h2>
<p>Provided actions 2, 3 and 4 yielded no errors (red crosses on the right hand pane) the pattern and spectra may now be plotted in Galaxy. Be aware that the visualisation zoom is fixed therefore it is recommended to plot 1 to 2ppm ranges of spectra to be able to see the annotations effectively. Also the height of the spectra are auto-scaled to maximum peak in visualisation range (so avoid plotting water or large peaks when trying to observed peaks near the baseline. </p>
<ul>
<li>Go to Plots </li>
<li>Plot NMR </li>
<li>NMR Spectra (the csv of your spectra prepared in 2.)</li>
<li>Interval is the region to show </li>
<li>plot bins</li>
<li>Select bin table as in 4.</li>
<li>Click Execute</li>
<li>Wait (if using more than 20 spectra this could take some time)</li>
<li>Click on eye icon once NMR Plot action (right hand panel) turns green</li>
</ul>
<h2 id="data%20analysis-data%20quality%20control-tamenmr-trouble-shooting">Trouble-shooting</h2>
<ul>
<li>If at any point the action panel on the right hand side has yellow actions these are in progress (need to wait for them to finish).</li>
<li>Once finished any failed processes are indicated in red – processes typically fail because the data is incorrectly formatted – check input data for correct dimensions and number of columns!</li>
</ul></section><h1 class='nav-section-title-end'>Ended: TameNMR</h1>
                        <h3 class='nav-section-title' id='section-topspin-quality-control'>
                            Topspin Quality Control <a class='headerlink' href='#section-topspin-quality-control' title='Permanent link'>↵</a>
                        </h3>
                        <section class="print-page" id="data%20analysis-data%20quality%20control-topspin%20quality%20control"><h1 id="data%20analysis-data%20quality%20control-topspin%20quality%20control-criteria-for-quality-control">Criteria for Quality Control</h1>
<p>Author: Marie Phelan
Revision Date: 2 November 2017</p>
<h2 id="data%20analysis-data%20quality%20control-topspin%20quality%20control-qc-criteria">QC Criteria</h2>
<table>
<thead>
<tr>
<th></th>
<th>QC</th>
<th>e.g.</th>
</tr>
</thead>
<tbody>
<tr>
<td>1</td>
<td>Shape and width of reference peak (usually TSP) Line width typically measured at half height</td>
<td><img alt="4_1_QC1" src="../Data%20Analysis/Data%20Quality%20Control/Topspin%20Quality%20Control/4_1_QC1.PNG" /></td>
</tr>
<tr>
<td>2</td>
<td>Baseline flat without curvature or sine wiggle.</td>
<td><img alt="4_1_QC2" src="../Data%20Analysis/Data%20Quality%20Control/Topspin%20Quality%20Control/4_1_QC2.PNG" /></td>
</tr>
<tr>
<td>3</td>
<td>Signal to noise as expected (check against representative spectrum)</td>
<td><img alt="4_1_QC3" src="../Data%20Analysis/Data%20Quality%20Control/Topspin%20Quality%20Control/4_1_QC3.PNG" /></td>
</tr>
<tr>
<td>4</td>
<td>Water suppression good. Narrow water signal (between 0.2 and 0.4 ppm wide). No baseline distortion beyond that range.</td>
<td><img alt="4_1_QC4" src="../Data%20Analysis/Data%20Quality%20Control/Topspin%20Quality%20Control/4_1_QC4.PNG" /></td>
</tr>
<tr>
<td>5</td>
<td>Phasing good throughout spectra</td>
<td><img alt="4_1_QC5" src="../Data%20Analysis/Data%20Quality%20Control/Topspin%20Quality%20Control/4_1_QC5.png" /></td>
</tr>
</tbody>
</table>
<h2 id="data%20analysis-data%20quality%20control-topspin%20quality%20control-topspin-shortcuts">Topspin Shortcuts</h2>
<table>
<thead>
<tr>
<th>Action</th>
<th align="center">Icon(s)</th>
<th>Text</th>
<th>Use</th>
</tr>
</thead>
<tbody>
<tr>
<td>Navigate browser</td>
<td align="center"></td>
<td></td>
<td>Click the circle to open folder</td>
</tr>
<tr>
<td>Increase spectra intensity</td>
<td align="center"><img alt="mult2" src="../Data%20Analysis/Data%20Quality%20Control/Topspin%20Quality%20Control/mult2.png" /><img alt="mult8" src="../Data%20Analysis/Data%20Quality%20Control/Topspin%20Quality%20Control/mult8.png" /></td>
<td></td>
<td></td>
</tr>
<tr>
<td>Decrease spectra intensity</td>
<td align="center"><img alt="div2" src="../Data%20Analysis/Data%20Quality%20Control/Topspin%20Quality%20Control/divide2.png" /><img alt="div8" src="../Data%20Analysis/Data%20Quality%20Control/Topspin%20Quality%20Control/divide8.png" /></td>
<td></td>
<td></td>
</tr>
<tr>
<td>Interactive intensity change</td>
<td align="center"><img alt="vexpand" src="../Data%20Analysis/Data%20Quality%20Control/Topspin%20Quality%20Control/vexpand.png" /></td>
<td></td>
<td>holding left mouse button over icon and dragging either up or down</td>
</tr>
<tr>
<td>View a specific ppm range of the spectrum</td>
<td align="center"><img alt="zoom_E_24" src="../Data%20Analysis/Data%20Quality%20Control/Topspin%20Quality%20Control/zoom_E_24.png" /></td>
<td></td>
<td></td>
</tr>
<tr>
<td>Undo selection of specific ppm range</td>
<td align="center"><img alt="undo4" src="../Data%20Analysis/Data%20Quality%20Control/Topspin%20Quality%20Control/undo4.png" /></td>
<td></td>
<td></td>
</tr>
<tr>
<td>View entire spectrum (no specific ppm range)</td>
<td align="center"></td>
<td></td>
<td></td>
</tr>
<tr>
<td>Pan along the spectrum</td>
<td align="center"></td>
<td></td>
<td></td>
</tr>
<tr>
<td>Spectrum can be raised or lowered</td>
<td align="center"></td>
<td></td>
<td>holding left mouse button over icon and dragging either up or down</td>
</tr>
<tr>
<td>Manual alignment</td>
<td align="center"><img alt="cal" src="../Data%20Analysis/Data%20Quality%20Control/Topspin%20Quality%20Control/calib_24.png" /></td>
<td>.cal</td>
<td>Then select peak and set ppm</td>
</tr>
<tr>
<td><u><strong>start overlay</strong></u></td>
<td align="center"><img alt="multi_spec" src="../Data%20Analysis/Data%20Quality%20Control/Topspin%20Quality%20Control/multi_spec.png" /></td>
<td>.md</td>
<td>This opens submenu for multidisplay mode</td>
</tr>
<tr>
<td>This opens a new set of icons</td>
<td align="center"></td>
<td></td>
<td></td>
</tr>
<tr>
<td>Reset spectra</td>
<td align="center"><img alt="ph_reset" src="../Data%20Analysis/Data%20Quality%20Control/Topspin%20Quality%20Control/ph_reset.png" /></td>
<td></td>
<td></td>
</tr>
<tr>
<td>Separate or collapse spectra</td>
<td align="center"></td>
<td></td>
<td></td>
</tr>
<tr>
<td>Difference spectrum</td>
<td align="center"><img alt="diff" src="../Data%20Analysis/Data%20Quality%20Control/Topspin%20Quality%20Control/diff.png" /></td>
<td></td>
<td></td>
</tr>
<tr>
<td>Remove selected spectra</td>
<td align="center"><img alt="delete_all" src="../Data%20Analysis/Data%20Quality%20Control/Topspin%20Quality%20Control/delete_all.png" /></td>
<td></td>
<td></td>
</tr>
<tr>
<td>Exit multi-display</td>
<td align="center"><img alt="return_24" src="../Data%20Analysis/Data%20Quality%20Control/Topspin%20Quality%20Control/return_24.png" /></td>
<td></td>
<td></td>
</tr>
<tr>
<td><strong><u>Advanced commands</u></strong></td>
<td align="center"></td>
<td></td>
<td></td>
</tr>
<tr>
<td>enter interactive phasing</td>
<td align="center"><img alt="phase_correction" src="../Data%20Analysis/Data%20Quality%20Control/Topspin%20Quality%20Control/phase_correction.png" /></td>
<td>.ph</td>
<td></td>
</tr>
<tr>
<td>This opens a new set of icons:</td>
<td align="center"></td>
<td></td>
<td></td>
</tr>
<tr>
<td>Zero order phase correction</td>
<td align="center"><img alt="ph0" src="../Data%20Analysis/Data%20Quality%20Control/Topspin%20Quality%20Control/ph0.png" /></td>
<td></td>
<td>Holding left mouse button over icon and dragging either up or down</td>
</tr>
<tr>
<td>Reset phasing</td>
<td align="center"><img alt="ph_reset" src="../Data%20Analysis/Data%20Quality%20Control/Topspin%20Quality%20Control/ph_reset.png" /></td>
<td></td>
<td></td>
</tr>
<tr>
<td>Invert the sign</td>
<td align="center"><img alt="ph180" src="../Data%20Analysis/Data%20Quality%20Control/Topspin%20Quality%20Control/ph180.png" /></td>
<td></td>
<td></td>
</tr>
<tr>
<td>Save return from interactive phase</td>
<td align="center"><img alt="ph_save_return" src="../Data%20Analysis/Data%20Quality%20Control/Topspin%20Quality%20Control/ph_save_return.png" /></td>
<td>.sret</td>
<td>Save changes</td>
</tr>
<tr>
<td>Exit phasing</td>
<td align="center"><img alt="ph_return" src="../Data%20Analysis/Data%20Quality%20Control/Topspin%20Quality%20Control/ph_return.png" /></td>
<td>.ret</td>
<td>Does <strong>NOT</strong> save changes</td>
</tr>
<tr>
<td>Baseline correction</td>
<td align="center"></td>
<td>absn</td>
<td></td>
</tr>
<tr>
<td>Re-process spectra</td>
<td align="center"></td>
<td>apk0.noe</td>
<td></td>
</tr>
<tr>
<td>Reference to glucose</td>
<td align="center"></td>
<td>glucoseref</td>
<td>For spectra of proteinous fluids</td>
</tr>
<tr>
<td>Reference to chloroform</td>
<td align="center"></td>
<td>chloroformref</td>
<td>For spectra of lipids</td>
</tr>
</tbody>
</table></section><h1 class='nav-section-title-end'>Ended: Topspin Quality Control</h1><h1 class='nav-section-title-end'>Ended: Data Quality Control</h1>
                        <h2 class='nav-section-title' id='section-statistical-analyses'>
                            Statistical Analyses <a class='headerlink' href='#section-statistical-analyses' title='Permanent link'>↵</a>
                        </h2>
                        
                        <h3 class='nav-section-title' id='section-pathway-analysis-in-metaboanalyst'>
                            Pathway Analysis in MetaboAnalyst <a class='headerlink' href='#section-pathway-analysis-in-metaboanalyst' title='Permanent link'>↵</a>
                        </h3>
                        <section class="print-page" id="data%20analysis-statistical%20analyses-pathway%20analysis%20in%20metaboanalyst"><h1 id="data%20analysis-statistical%20analyses-pathway%20analysis%20in%20metaboanalyst-pathway-analysis-in-metaboanalyst">Pathway Analysis in MetaboAnalyst</h1>
<p>Authors: Marie Phelan &amp; Rudi Grosman
Revision Date:2 nd November 2017</p>
<p>Go to MetaboAnalyst website (www.metaboanalyst.ca).</p>
<p>Click on ‘click here to start’, this will take you to the analysis selection page.</p>
<p>In the analysis selection page select ‘Pathway Analysis’.</p>
<p>In the input window enter the metabolites of interest.Makes sure you select the input type (available options are compound name, KEGG ID and HMDB ID).</p>
<p>If using compound names as input there is a name matching function that checks the names against the possible matches in the databse. In these cases the entry will be highlighted.</p>
<p>A yellow highlighted entry indicates an issue - click ‘View’ which will pop-up a window for possible entries. If an entry is highlighted with red, it means there is no match in the database.</p>
<p>Tick the correct compound from the list (select ‘None of the above’ if it is not listed). Then click ‘OK’.</p>
<p>Once the correct input selected MetaboAnalyst will fill the empty cells.</p>
<p>Once the input list has been checked (and corrected if necessary) click ‘submit’.</p>
<p>Select the organism that you want to perform pathway analysis on. The square brackets represent number of pathways available for the organisms. When the organism is selected scroll down.</p>
<p>Select <strong>Over Representation Analysis</strong> type and the <strong>Pathway Topology Analysis</strong> type (details can be found in the MetaboAnalyst tutorials). Select the compounds to be used and click ‘Submit’.</p>
<p>Each point on the <strong>Pathway Overview</strong> graph represents a pathway and when clicked on it will show the pathway in the next window. Red highlighted codes are the input metabolites. This section is followed by the list of pathways and their scores.</p>
<p>To download the results, click ‘Submit’ which will redirect to the download page.</p>
<p>On the download page either download individual sections or the whole analysis in a .zip file. To download everything select ‘Download.zip’.</p>
<h2 id="data%20analysis-statistical%20analyses-pathway%20analysis%20in%20metaboanalyst-references">References</h2>
<ol>
<li>Xia, J., Sinelnikov, I., Han, B., and Wishart, D.S. (2015) <a href="http://doi.org/10.1093/nar/gkv380">MetaboAnalyst 3.0 - making metabolomics more meaningful</a> . Nucl. Acids Res. 43, W251-257. </li>
<li>Xia, J. and Wishart, D.S. (2011) <a href="http://doi.org/10.1038/nprot.2011.319">Web-based inference of biological patterns, functions and pathways from metabolomic data using MetaboAnalyst</a> Nature Protocols 6 (6), 743-760. </li>
<li>Xia, J. and Wishart, D.S. (2011) <a href="http://doi.org/10.1002/0471250953.bi1410s34">Metabolomic data processing, analysis, and interpretation using MetaboAnalyst</a> Current Protocols in Bioinformatics, 14.10.1-14.10.48.</li>
<li>Xia, J., Broadhurst, D., Wilson, M. and Wishart, D. (2013) <a href="http://doi.org/10.1007/s11306-012-0482-9">Translational biomarker discovery in clinical metabolomics: an introductory tutorial</a>. Metabolomics, 9, 280-299. (biomarker analysis)</li>
<li>Xia, J. and Wishart, D.S. (2010) <a href="http://doi.org/10.1093/bioinformatics/btq418">MetPA: a web-based metabolomics tool for pathway analysis and visualization</a>. Bioinformatics, 26, 2342-2344 (metabolic pathway analysis)</li>
<li>Xia, J. and Wishart, D.S. (2010) <a href="http://doi.org/10.1093/nar/gkq329">MSEA: a web-based tool to identify biologically meaningful patterns in quantitative metabolomic data</a>. Nucleic Acids Research, 38, W71-77 (metabolite set enrichment analysis)</li>
</ol>
<p>For MetaboAnalyst tutorials visit: http://www.metaboanalyst.ca/faces/docs/Tutorial.xhtml</p>
<h2 id="data%20analysis-statistical%20analyses-pathway%20analysis%20in%20metaboanalyst-alternative-software">Alternative Software</h2>
<p>MetExplore – tutorials available online</p>
<h2 id="data%20analysis-statistical%20analyses-pathway%20analysis%20in%20metaboanalyst-alternative-pathway-resources">Alternative Pathway Resources</h2>
<p>HMDB1-3 Human metabolome database www.hmdb.ca SMPDB4-5 Small molecule pathways smpdb.ca KEGG6-10 Metabolites, genes and proteins pathway database www.kegg.jp </p>
<ol>
<li>Wishart DS, et al HMDB: the Human Metabolome Database (2007) DOI: <a href="https://doi.org/10.1093/nar/gkl923">10.1093/nar/gkl923</a></li>
<li>Wishart DS, et al HMDB: a knowledgebase for the human metabolome (2009) DOI: <a href="https://doi.org/10.1093/nar/gkn810">10.1093/nar/gkn810</a></li>
<li>Wishart DS, et al HMDB 3.0 – The Human Metabolome Database in 2013 (2013) DOI: <a href="https://doi.org/10.1093/nar/gks1065">10.1093/nar/gks1065</a></li>
<li>Wishart DS, et al SMPDB: The Small Molecule Pathway Database DOI: <a href="https://doi.org/10.1093/nar/gkp1002">10.1093/nar/gkp1002</a></li>
<li>Jewison T, et al SMPDB 2.0: Big Improvements to the Small Molecule Pathway Database DOI: <a href="https://doi.org/10.1093/nar/gkt1067">10.1093/nar/gkt1067</a> </li>
<li>Hashimoto, K.et al Comprehensive analysis of glycosyltransferases in eukaryotic genomes for structural and functional characterization of glycans (2009). DOI:<a href="https://doi.org/10.1016/j.carres.2009.03.001">10.1016/j.carres.2009.03.001</a></li>
<li>Hashimoto, K. et al The repertoire of desaturases and elongases reveals fatty acid variations in 56 eukaryotic genomes (2008). DOI:<a href="https://doi.org/10.1194/jlr.M700377-JLR200">10.1194/jlr.M700377-JLR200</a></li>
<li>Minowa, Y. et al Comprehensive analysis of distinctive polyketide and nonribosomal peptide structural motifs encoded in microbial genomes (2007). DOI:<a href="https://doi.org/10.1016/j.jmb.2007.02.099">10.1016/j.jmb.2007.02.099</a></li>
<li>Hattori, M. et al Development of a chemical structure comparison method for integrated analysis of chemical and genomic information in the metabolic pathways (2003). DOI:<a href="https://doi.org/10.1021/ja036030u">10.1021/ja036030u</a></li>
<li>Hattori, M. et al SIMCOMP/SUBCOMP: chemical structure search servers for network analyses (2010). DOI:<a href="https://doi.org/10.1093/nar/gkq367">10.1093/nar/gkq367</a></li>
</ol></section><h1 class='nav-section-title-end'>Ended: Pathway Analysis in MetaboAnalyst</h1>
                        <h3 class='nav-section-title' id='section-statistical-analysis-in-metaboanalyst'>
                            Statistical Analysis in MetaboAnalyst <a class='headerlink' href='#section-statistical-analysis-in-metaboanalyst' title='Permanent link'>↵</a>
                        </h3>
                        <section class="print-page" id="data%20analysis-statistical%20analyses-statistical%20analysis%20in%20metaboanalyst"><h1 id="data%20analysis-statistical%20analyses-statistical%20analysis%20in%20metaboanalyst-a-selection-of-statistical-analyses-in-metaboanalyst">A selection of statistical analyses in MetaboAnalyst</h1>
<p>Authors: Marie Phelan &amp; Eva Caamaño-Gutiérrez
Revision Date: 2 November 2017</p>
<h2 id="data%20analysis-statistical%20analyses-statistical%20analysis%20in%20metaboanalyst-part-1-submitting-data">Part 1 Submitting Data</h2>
<p>Open MetaboAnalyst at www.metaboanalyst.ca </p>
<ol>
<li>Click on “click here to start”</li>
<li>Upload data for Statistical Analysis</li>
<li>Peak intensity table works for AMIX exported bucket tables</li>
<li>Upload your file here in .csv format</li>
<li>Submit- errors will appear if file is not correctly formatted <strong>Edit the .csv file output from AMIX for correct upload to MetaboAnalyst.</strong></li>
</ol>
<p>Some of the editing needed to make the file compatible with MetaboAnalyst involves the incorporation of some key information for the analysis, for example:</p>
<ul>
<li>Open bucket file in spreadsheet software.</li>
<li>
<p>Add a new column after the name of each spectrum (See example spreadsheet on Metaboanalyst). The first column (name) corresponds to the sample identity kept from the spectrometer. The second column (group) contains the key information for future grouping and testing.</p>
</li>
<li>
<p>Data check</p>
</li>
</ul>
<p>Once you have uploaded your file the data integrity check screen will appear. In this step you can decide how to deal with missing values if applicable. There is functionality to choose to ignore variables with over a percentage of samples missing or to use an algorithm to estimate those missing values. However, most of the data produced via the tools presented in this workshop will not require this step and we can click “skip”. </p>
<ol>
<li>Data filtering</li>
</ol>
<p>This functionality is paramount when trying to reduce dimensionality of the data for computational purposes (many variables=longer computational time) and to reduce unwanted variance, specially remove outliers. However, if you have less than 2000 variables/bucket/metabolites and you are unsure on how to filter your data, you can choose to not filter your data (“none).</p>
<ol>
<li>Normalisation Options</li>
</ol>
<p>The aim of <strong>normalisation</strong> is to make <u>samples</u> comparable (for example to reduce variation due to dilution effects, ‘pipetters’ or ). Some examples include normalisation by an internal standard e.g. a metabolite that is at constant concentration between treatments or If looking at a time series for example we could also normalise all other samples by the time zero. Some data transformations also aid in some analyses. For example log transformation is usually helpful in biological data that which distribution is skewed towards positive values. Each dataset is different and requires its own evaluation.</p>
<p><strong>Scaling</strong> is a way of making the <u>variables</u> in the data more similar in range. If the variables are of different scale some linear analysis methods (like PCA) will tend to be influenced more by larger variables skewing the results. <em>Pareto</em> scaling is a popular choice in metabolomics studies and is available in MetaboAnalyst.</p>
<p>When you have finished, click Submit to view the data normalisation (see Figure below). After the transformations the data is centred around zero and is of similar scale, with not variables overweighting by solely their nature instead of the effect the treatments have on them. It is ready for further analysis, press next</p>
<p>Click <strong>Submit</strong> to view the data normalisation.</p>
<p>After the transformations the data is centred around zero and is of similar scale. It is ready for further analysis. <strong>Submit</strong> </p>
<p>The data is now prepared for application of the numerous statistical tests available in MetaboAnalyst.</p>
<p>Different tests are available depending on how many cohorts (GROUPS) you have. In this example there are more than Two cohorts, Thus multivariate tests such as ANOVA and available and twofold tests such as T-Test and Fold Changes are not.</p>
<p>## Part 2 Statistical Analysis</p>
<p>You can now explore the statistical tests MetaboAnalyst offers. Different tests are available depending on how many cohorts (CLASS/GROUP) you have. We have &gt;2 cohorts, so multivariate tests such as ANOVA are available but not tests for only two variables such as t-test.</p>
<p>## Part 2a Statistical Analysis – ANOVA</p>
<ol>
<li>
<p>Choose parameters &amp; explore data</p>
<ol>
<li>Choose a p-value threshold </li>
<li>Choose post hoc analysis technique (method to do cohort comparisons on significant points (both Fisher and Tukey) and correct for false positives (Tukey)). </li>
<li>Red data points are significantly different </li>
<li>Dotted line represents the p-value cut-off </li>
<li>Blue data points are not significantly different </li>
<li>Click on each data point to get a box plot showing the different in intensity of the peak at that ppm between the samples. </li>
<li>Clicking on the table shows the post-hoc table (see next page)</li>
<li>Click on the paint palette to download images</li>
</ol>
</li>
<li>
<p>Post-hoc tables are available from the ANOVA</p>
<p>The Table only reports those buckets that are significantly different between cohorts, in the post-hoc tests section it can be seen between which cohorts the bucket is significantly different.</p>
<p>Clicking on each of the buckets (1.914Lys, for example) brings up a box plot of the differences of that bucket between cohorts.</p>
<p>The table is downloadable as well as the box plots.</p>
</li>
</ol>
<p>## Part 2b Heat maps</p>
<p>Heat maps are a very useful visualisation method. For a quick overview of how the groups are different between them in relation to the bucket signal.</p>
<p>## Part 2c Statistical Analysis – PCA</p>
<p>To launch PCA analysis select PCA from side menu (red arrow)</p>
<p>1) In the Overview you can visualise your data to five principle components (PC's) with a pairwise score plot for each PC against every other PC</p>
<p>2) Scree plot - shows % explained variance both per PC and cumulatively</p>
<p>3) 2D scores plot - shows scores for annotated samples (including 95% confidence elipsoid) for specified PCs</p>
<p>4) 3D scores plot - shows moveable cube with 3 PCs plotted against each other. Hover mouse over each point to get specific sample information.</p>
<p>5) Loadings plot - shows metabolite bucket weightings responsible for specified PCs. Hover the mouse over individual point to obtain specific bucket weighting. Click on specific point to see a boxplot representation.</p>
<p>6) Biplot – combines sample score to most influential buckets.</p>
<p>## Part 2d Statistical Analysis – PLS-DA and OPLS-DA</p>
<p>Partial Least Squares-Discriminant Analysis (PLS-DA) and Orthogonal Projections to Latent Structures (O-PLS) are supervised multivariate analyses. Similar to PCA the PLSDA algorithm can be launched by clicking the tab. The output will be akin to PCA’s but with some key differences. Main similarities are the score plots of the PLS components and the loadings. Similarly than with PCA, latent variables arise that provide new projections (scores) and to which the original variables have certain contribution (loadings).</p>
<p>Because <u>over-fitting</u> is a very important issue, there are main methods to try to control it implemented in MetaboAnalyst. These provide statistical insight on how many variables to choose to create the model. The algorithm implemented within MetaboAnalyst should automatically select chosen the best fit for the data.</p>
<p>1) <u>Cross Validation</u></p>
<p>In the cross-validation tab several parameters can be selected such as the performance measure. The optimal is set to Q<sup>2</sup> , which is the cross-validated sum of squares captured by the model.</p>
<p>Another method to control over fitting is permutation test. These rely on assessing how likely is to get the separation obtained by the model randomly. For that purpose, the labels (groups/cohorts) are randomised/unordered and the model is made again. With random labels, no significant differences should be found between groups. This process is repeated iteratively 100, 1000 or 2000 times (in MetaboAnalyst) and allows deriving the probability of separating the original groups just by chance. A low p value indicates that the chances for the grouping to be random are low. </p>
<p>2) <u>Importance Features</u></p>
<p>In the tab “Imp. Features”, there are two variable importance measures in PLS-DA, that can be downloaded by pressing on the little sheet icon.</p>
<p>The first, Variable Importance in Projection (VIP), is a weighted sum of squares of the PLS loadings taking into account the amount of explained Y-variation, in each dimension this can be interpreted as a measure of the contribution <u>or importance of a variable to the model</u>. Since VIP scores are calculated for each component when more than 1 component are used to calculate the feature importance the average of the VIP scores are used.</p>
<p>The other importance measure is based on the weighted sum of PLS-regression. The weights are a function of the reduction of the sums of squares across the number of PLS components. The coefficient of each feature will be different depending on which group you want to predict. The average of the feature coefficients are used as the overall coefficient-based importance.</p>
<p>3) <u>OPLS-DA</u> </p>
<p>The last feature that has been implemented in MetaboAnalyst is OPLS. Documentation is still limited but it is note worthy the visualisation improvement that might provide with respect to PLSDA (see below, groups separation is more obvious with only two dimensions).</p>
<h2 id="data%20analysis-statistical%20analyses-statistical%20analysis%20in%20metaboanalyst-part-3-modifying-and-extracting-the-data">Part 3 Modifying and extracting the data</h2>
<ol>
<li>At any time the left hand menu allows modification of the dataset under investigation. Selecting upload, processing and normalisation will return to the corresponding process. Selecting Statistics opens a list of statistical analyses available in MetaboAnalyst for the particular dataset (dependent on number of groups etc.)</li>
</ol>
<h2 id="data%20analysis-statistical%20analyses-statistical%20analysis%20in%20metaboanalyst-part-4-download">Part 4 Download</h2>
<p>In the Download section all or part of the analysis results can be downloaded.</p>
<p>Clicking on the Download.zip downloads everything in one zipped file. This is sometimes a good option as MetaboAnalyst will time out after a period of inactivity require re-inputting of the data.</p>
<p>Logging out ends your session and the data will no longer be available so make sure you have downloaded everything you need before you do this.</p>
<h2 id="data%20analysis-statistical%20analyses-statistical%20analysis%20in%20metaboanalyst-more-information">More Information</h2>
<p>Jianguo Xia &amp; David S Wishart “Web-based inference of biological patterns, functions and pathways from metabolomic data using MetaboAnalyst” Nature Protocols 6, 743-760 (2011) doi:<a href="https://doi.org/10.1038/nprot.2011.319">10.1038/nprot.2011.319</a></p></section><h1 class='nav-section-title-end'>Ended: Statistical Analysis in MetaboAnalyst</h1><h1 class='nav-section-title-end'>Ended: Statistical Analyses</h1><h1 class='nav-section-title-end'>Ended: Data Analysis</h1>
                        <h1 class='nav-section-title' id='section-miscellaneous'>
                            Miscellaneous <a class='headerlink' href='#section-miscellaneous' title='Permanent link'>↵</a>
                        </h1>
                        
                        <h2 class='nav-section-title' id='section-tube-washing'>
                            Tube washing <a class='headerlink' href='#section-tube-washing' title='Permanent link'>↵</a>
                        </h2>
                        <section class="print-page" id="miscellaneous-tube%20washing"><h1 id="miscellaneous-tube%20washing-tube-washing-for-nmr-metabolomics">Tube Washing for NMR Metabolomics</h1>
<p>The purpose of this protocol is to ensure NMR tubes are washed to a standard compatible with NMR metabolomics – deviations from this protocol can result in contamination of samples with detergents and solvents that will make the resultant data unusable.</p>
<h2 id="miscellaneous-tube%20washing-soakwash">Soak/Wash</h2>
<ol>
<li>Soak NMR tubes and caps with decon90 from squirt bottle (10% decon90)</li>
<li>Ensure all tubes and caps are completely filled with decon90 and let it soak overnight</li>
<li>Ensure all tubes' opening are facing upright so that decon90 does not leak out</li>
</ol>
<p><img alt="A picture containing text, kitchen appliance  Description automatically generated" src="../Miscellaneous/Tube%20washing/Decon.png" /></p>
<p><strong>Figure 1:</strong> Decon90 storage and squeeze bottle.</p>
<h2 id="miscellaneous-tube%20washing-after-soaking">After soaking</h2>
<ol>
<li>Rinse out decon90 from the NMR tubes using glass pipette (rinse does not need to be thorough, as long as water displaces some decon90).</li>
<li>Set up water pump which helps to wash the NMR tubes thoroughly</li>
</ol>
<p><img alt="A picture containing text, different, items, various  Description automatically generated" src="../Miscellaneous/Tube%20washing/PumpSetup.png" /></p>
<p><strong>Figure 2:</strong> set up for the water pump. a) location for multitube adaptor b) fully insert 10 upended tubes over the green straws within the adaptor c) make a seal between the adaptor and the vacuum flask d) attach the pump tubing to the vacuum flask outlet.</p>
<ol>
<li>Ensure 5 litres of water is added to the adaptor and pumped through the tubes (in order to do this you need to empty the bottom chamber 5 times – do not load more than 1 litre before emptying the bottom chamber).</li>
</ol>
<blockquote>
<p><mark>!WARNING!</mark> avoid getting water into the trap (this prevents water being drawn into the electric pump and thus electrical failure).</p>
</blockquote>
<h2 id="miscellaneous-tube%20washing-washing-caps">Washing caps</h2>
<ol>
<li>
<p>Remove decon90 from the beaker the caps are in and Rinse the caps in another beaker thoroughly with miliQ water, preferably 7-8 times</p>
</li>
<li>
<p>After washing, dry using the air jet to dry the tubes and caps</p>
</li>
<li>
<p>After drying the tubes using the air jet, look at the tubes under light to check for dirt on the outer surface of the tubes. Clean when necessary.</p>
</li>
<li>
<p>Keep the clean tubes and caps in a dry beaker.</p>
</li>
</ol>
<h2 id="miscellaneous-tube%20washing-tube-drying">Tube Drying</h2>
<ol>
<li>
<p>Use the airflow from the piped gas line (bottom wall-mounted tap behind the ELGA water purifier). </p>
</li>
<li>
<p>Angle the tap to ~5 o'clock to ensure the air-flow is not too vigorous.</p>
</li>
<li>
<p>Dry each tube using white lint-free tissue to recover any water droplets by upending the tube over the tube dry station to the right of sink (for 3mm tubes carefully attach a glass Pasteur pipette to the top of the drying rod using the rubber attachment provided).</p>
</li>
<li>
<p>Turn off the airflow after use.</p>
</li>
</ol>
<blockquote>
<p><mark>!WARNING!</mark> Failure to turn off the airflow prevents air compressor being available for the NMR spectrometers (for hydraulics of auto samplers and lifting/lowering samples into the magnet so this step is ESSENTIAL to maintain Facility operation.</p>
</blockquote></section><h1 class='nav-section-title-end'>Ended: Tube washing</h1><h1 class='nav-section-title-end'>Ended: Miscellaneous</h1>
                        <h1 class='nav-section-title' id='section-resources'>
                            Resources <a class='headerlink' href='#section-resources' title='Permanent link'>↵</a>
                        </h1>
                        
                        <h2 class='nav-section-title' id='section-background-reading'>
                            Background Reading <a class='headerlink' href='#section-background-reading' title='Permanent link'>↵</a>
                        </h2>
                        <section class="print-page" id="resources-background%20reading"><h1 id="resources-background%20reading-background-reading">Background Reading</h1>
<p>Author: Marie Phelan
Revision Date: 1 March 2019</p>
<div  class="mermaid">graph TB

subgraph <br>
AA((Study<br>Design)) --> B((Study<br>Design))
AA --> C((Sample<br>Collection))
AA --> D((Sample<br>Preparation))

AB((Data<br>Analysis)) --> E((Quality<br>Control))
AB --> F((Data<br>Formatting))
AB --> G((Common<br>Statistics))
AB --> H((Metabolite<br>Annotation<br>& Identities))

AC((Biological<br>Contextualisation)) --> I((Data<br>Deposition))
AC --> J((Link to<br>Biological<br>Pathway))
end

</div>
<h2 id="resources-background%20reading-study-design">Study Design</h2>
<p>Metabolic profiling, metabolomic and metabonomic procedures for NMR spectroscopy of urine, plasma, serum and tissue extracts.
Beckonert O, Keun HC, Ebbels TM, Bundy J, Holmes E, Lindon JC, Nicholson JK.
Nat Protoc. 2007;2(11):2692-703.
https://www.ncbi.nlm.nih.gov/pubmed/18007604</p>
<p>NMR Metabolomics: A Comparison of the Suitability of Various Commonly used National Health Service Blood Collection Tubes
MM Phelan, LY Lian.
Current Metabolomics Volume 4 , Issue 1 , 2016 78 - 81
http://www.eurekaselect.com/135553</p>
<p>Using an NMR metabolomics approach to investigate the pathogenicity of amyloid-beta and alpha-synuclein
M. M. Phelan, E. Caamaño-Gutiérrez, M. S. Gant, R. X. Grosman, and J. Madine
Metabolomics. 2017; 13(12): 151.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5661010/</p>
<p>Transcriptomic and metabolomic data integration
Rachel Cavill Danyel Jennen Jos Kleinjans Jacob Jan Briedé
Briefings in Bioinformatics, Volume 17, Issue 5, 1 September 2016, Pages 891-901
https://academic.oup.com/bib/article/17/5/891/2262240</p>
<p>High-throughput serum NMR metabonomics for cost-effective holistic studies on systemic metabolism
Soininen P, Kangas AJ, Würtz P, Tukiainen T, Tynkkynen T, Laatikainen R, Järvelin MR, Kähönen M, Lehtimäki T, Viikari J, Raitakari OT, Savolainen MJ, Ala-Korpela M.
Analyst. 2009 Sep;134(9):1781-5. doi: 10.1039/b910205a. Epub 2009 Jul 30.
https://www.ncbi.nlm.nih.gov/pubmed/19684899</p>
<p>Databases and Software for NMR-Based Metabolomics
Ellinger JJ, Chylla RA, Ulrich EL, Markley JL.
Curr Metabolomics. 2013;1(1). doi: 10.2174/2213235X11301010028
https://www.ncbi.nlm.nih.gov/pubmed/24260723</p>
<p>Current practice of liquid chromatography-mass spectrometry in metabolomics and metabonomics
Gika HG, Theodoridis GA, Plumb RS, Wilson ID
J Pharm Biomed Anal. 2014 Jan;87:12-25. doi: 10.1016/j.jpba.2013.06.032. Epub 2013 Jul 17
https://www.ncbi.nlm.nih.gov/pubmed/23916607</p>
<p>Global metabolic profiling procedures for urine using UPLC-MS. Want EJ, Wilson ID, Gika H, Theodoridis G, Plumb RS, Shockcor J, Holmes E, Nicholson JK. Nat Protoc. 2010 Jun;5(6):1005-18. doi: 10.1038/nprot.2010.50. https://www.ncbi.nlm.nih.gov/pubmed/20448546</p>
<p>Proposed minimum reporting standards for chemical analysis Chemical Analysis Working Group (CAWG) Metabolomics Standards Initiative (MSI) Lloyd W. Sumner, Alexander Amberg, Dave Barrett, Michael H. Beale, Richard Beger, Clare A. Daykin, Teresa W.-M. Fan, Oliver Fiehn, Royston Goodacre, Julian L. Griffin, Thomas Hankemeier, Nigel Hardy, James Harnly, Richard Higashi, Joachim Kopka, Andrew N. Lane, John C. Lindon, Philip Marriott, Andrew W. Nicholls, Michael D. Reily, John J. Thaden, and Mark R. Viant Metabolomics. 2007 Sep; 3(3): 211\u2013221. https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3772505/</p>
<p>The role of reporting standards for metabolite annotation and identification in metabolomic studies. Salek RM, Steinbeck C, Viant MR, Goodacre R, Dunn WB. Gigascience. 2013 Oct 16;2(1):13. doi: 10.1186/2047-217X-2-13. https://www.ncbi.nlm.nih.gov/pubmed/24131531</p>
<p>Quality assurance and quality control processes: Summary of a metabolomics community questionnaire Warwick Dunn, David Broadhurst, Art Edison, Mark Viant, Daniel W. Bearden, Richard Beger, Claude Guillou https://www.nist.gov/publications/quality-assurance-and-quality-control-processes-summary-metabolomics-community</p>
<p>Data Analysis Proposed minimum reporting standards for chemical analysis Chemical Analysis Working Group (CAWG) Metabolomics Standards Initiative (MSI)
Lloyd W. Sumner,corresponding author Alexander Amberg, Dave Barrett, Michael H. Beale, Richard Beger, Clare A. Daykin, Teresa W.-M. Fan, Oliver Fiehn, Royston Goodacre, Julian L. Griffin, Thomas Hankemeier, Nigel Hardy, James Harnly, Richard Higashi, Joachim Kopka, Andrew N. Lane, John C. Lindon, Philip Marriott, Andrew W. Nicholls, Michael D. Reily, John J. Thaden, and Mark R. Viant
Metabolomics. 2007 Sep; 3(3): 211-221
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3772505/</p>
<p>The role of reporting standards for metabolite annotation and identification in metabolomic studies
Salek RM, Steinbeck C, Viant MR, Goodacre R, Dunn WB1
Gigascience. 2013 Oct 16;2(1):13. doi: 10.1186/2047-217X-2-13
https://www.ncbi.nlm.nih.gov/pubmed/24131531</p>
<p>Quality assurance and quality control processes: Summary of a metabolomics community questionnaire
Warwick Dunn, David Broadhurst, Art Edison, Mark Viant, Daniel W. Bearden, Richard Beger, Claude Guillou
https://www.nist.gov/publications/quality-assurance-and-quality-control-processes-summary-metabolomics-community</p>
<p>Global metabolic profiling procedures for urine using UPLC-MS
Want EJ, Wilson ID, Gika H, Theodoridis G, Plumb RS, Shockcor J, Holmes E, Nicholson JK
Nat Protoc. 2010 Jun;5(6):1005-18. doi: 10.1038/nprot.2010.50
https://www.ncbi.nlm.nih.gov/pubmed/20448546</p>
<p>State-of-the art data normalization methods improve NMR-based metabolomic analysis
Kohl, S. M., M. S. Klein, J. Hochrein, P. J. Oefner, R. Spang, W. Gronwald (2012)
Metabolomics 8, 146-160
https://www.ncbi.nlm.nih.gov/pubmed/22593726</p>
<p>Comparing groups for statistical differences: how to choose the right statistical test?
M Marusteri, V Bacarea
Biochemia Medica 2010;20(1):15-32
http://www.biochemia-medica.com/content/comparing-groups-statistical-differences-how-choose-right-statistical-test</p>
<p>Web-based inference of biological patterns, functions and pathways from metabolomic data using MetaboAnalyst
Jianguo Xia &amp; David S Wishart
Nature Protocols 6, 743-760 (2011) doi:10.1038/nprot.2011.319
https://www.nature.com/articles/nprot.2011.319</p>
<p>Multivariate Analysis in Metabolomics
Bradley Worley &amp; Robert Powers
Curr Metabolomics. 2013; 1(1): 92-107. doi: 10.2174/2213235X11301010092
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4465187/</p>
<h2 id="resources-background%20reading-biological-contextualisation">Biological Contextualisation</h2>
<p>High-throughput serum NMR metabonomics for cost-effective holistic studies on systemic metabolism
Soininen P, Kangas AJ, Würtz P, Tukiainen T, Tynkkynen T, Laatikainen R, Järvelin MR, Kähönen M, Lehtimäki T, Viikari J, Raitakari OT, Savolainen MJ, Ala-Korpela M.Soininen
Analyst. 2009 Sep;134(9):1781-5 Analyst. 2009 Sep;134(9):1781-5. doi: 10.1039/b910205a. Epub 2009 Jul 30
https://www.ncbi.nlm.nih.gov/pubmed/19684899</p>
<p>MetaboLights-- an open-access general-purpose repository for metabolomics studies and associated meta-data
Haug et al Nucl. Acids Res. (2013) doi: 10.1093/nar/gks1004
https://academic.oup.com/nar/article/41/D1/D781/1050654</p>
<p>Databases and Software for NMR-Based Metabolomics
Ellinger JJ1, Chylla RA, Ulrich EL, Markley JL.
Curr Metabolomics. 2013;1(1). doi: 10.2174/2213235X11301010028
https://www.ncbi.nlm.nih.gov/pubmed/24260723</p>
<p>The CCPN Metabolomics Project: a fast protocol for metabolite identification by 2D-NMR
Chignola F1, Mari S, Stevens TJ, Fogh RH, Mannella V, Boucher W, Musco G.
Bioinformatics. 2011 Mar 15;27(6):885-6. doi: 10.1093/bioinformatics/btr013. Epub 2011 Jan 6.
https://www.ncbi.nlm.nih.gov/pubmed/21216771</p>
<p>MSEA: a web-based tool to identify biologically meaningful patterns in quantitative metabolomic data
Jianguo Xia David S. Wishart
Nucleic Acids Research, Volume 38, Issue suppl_2, 1 July 2010, Pages W71–W77
https://doi.org/10.1093/nar/gkq329 https://academic.oup.com/nar/article/38/suppl_2/W71/1101310</p>
<p>NMR Spectroscopy for Metabolomics and Metabolic Profiling
Cynthia K. Larive, Gregory A. Barding Jr. and Meredith M. Dinges
Anal. Chem., 2015, 87 (1), pp 133–146 DOI: 10.1021/ac504075g
https://pubs.acs.org/doi/abs/10.1021/ac504075g</p>
<p>HMDB: HMDB: the Human Metabolome Database
Wishart DS, et al
Nucleic Acids Res. 2007 35(Database issue): D521–D526 DOI: 10.1093/nar/gkl923
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1899095/</p>
<p>HMDB: a knowledgebase for the human metabolome
Wishart DS, et al
Nucleic Acids Res. 2009;37(Database issue):D603-10 DOI: 10.1093/nar/gkn810
https://www.ncbi.nlm.nih.gov/pubmed/18953024</p>
<p>HMDB 3.0 – The Human Metabolome Database in 2013
Wishart DS, et al
Nucleic Acids Res. 2013;41(Database issue):D801-7 DOI: 10.1093/nar/gks1065
https://www.ncbi.nlm.nih.gov/pubmed/23161693</p>
<p>HMDB 4.0: the human metabolome database for 2018
Wishart DS et al
Nucleic Acids Res. 2018 Jan 4;46(D1):D608-D617. doi: 10.1093/nar/gkx1089.
https://www.ncbi.nlm.nih.gov/pubmed/29140435</p>
<p>MetaboAnalyst: Web-based inference of biological patterns, functions and pathways from metabolomic data using MetaboAnalyst
Jianguo Xia &amp; David S Wishart
Nature Protocols 6, 743–760 (2011) doi:10.1038/nprot.2011.319
https://www.nature.com/articles/nprot.2011.319</p>
<p>Using MetaboAnalyst 3.0 for Comprehensive Metabolomics Data Analysis.
Xia J, Wishart DS
Current Protocols in Bioinformatics 2016 55, p141
https://www.ncbi.nlm.nih.gov/pubmed/27603023</p>
<p>KEGG: Development of a chemical structure comparison method for integrated analysis of chemical and genomic information in the metabolic pathways (2003)
Hattori, M. et al
J. Am. Chem. Soc., 2003, 125 (39), pp 11853–11865 DOI:10.1021/ja036030u
https://pubs.acs.org/doi/abs/10.1021/ja036030u</p>
<p>SIMCOMP/SUBCOMP: chemical structure search servers for network analyses
Hattori, M. et al
Nucleic Acids Research, 38, Issue suppl_2, 2010, W652–W656 DOI:10.1093/nar/gkq367
https://academic.oup.com/nar/article/38/suppl_2/W652/1103855</p></section><h1 class='nav-section-title-end'>Ended: Background Reading</h1><h1 class='nav-section-title-end'>Ended: Resources</h1>
                        <h1 class='nav-section-title' id='section-sample-preparation'>
                            Sample Preparation <a class='headerlink' href='#section-sample-preparation' title='Permanent link'>↵</a>
                        </h1>
                        
                        <h2 class='nav-section-title' id='section-blood'>
                            Blood <a class='headerlink' href='#section-blood' title='Permanent link'>↵</a>
                        </h2>
                        <section class="print-page" id="sample%20preparation-blood"><h1 id="sample%20preparation-blood-plasma-and-serum">Plasma and Serum</h1>
<p>Author: Marie Phelan
Revision Date: 4 October 2016</p>
<h2 id="sample%20preparation-blood-sample-preparation">Sample Preparation</h2>
<p>Avoid citrate and other added stabilizers since they give additional signals in the NMR spectra. Also, collection tubes should be avoided, which use gel to separate blood cells from plasma.</p>
<h3 id="sample%20preparation-blood-plasma">Plasma</h3>
<ul>
<li>Collect ~0.80 ml into Heparin coated tubes (e.g., Li-Hep monovette) to give plasma,
use standard site-specific procedures for blood collection.</li>
</ul>
<h3 id="sample%20preparation-blood-sera">Sera</h3>
<ul>
<li>Collect ~0.80 ml into uncoated tubes (e.g., Serum Z monovette) stand upright until
coagulation - use standard site-specific procedures for blood collection. The time
before separation of blood cells should not exceed 30 min.</li>
</ul>
<h3 id="sample%20preparation-blood-both">Both</h3>
<ul>
<li>Centrifuge samples at 1,600g for 15 min at 4<span>&deg;</span>C.</li>
<li>Store supernatant at –80<span>&deg;</span>C until analyzed. Samples may be stored for up to 9 months <a href="https://pubs.acs.org/doi/10.1021/ac051972y">Teahan, O. et al. Anal. Chem. 78, 4307–4318 (2006)</a>.</li>
</ul>
<h2 id="sample%20preparation-blood-immediately-prior-to-nmr-acquisition-at-310k">Immediately prior to NMR acquisition at 310K</h2>
<ul>
<li>Add to 500μl of Plasma/Sera:<ul>
<li>100μl 1M Sodium Phosphate pH 7.4 in D<sub>2</sub>O</li>
<li>400μl H<sub>2</sub>O</li>
<li>2.5μl NaN<sub>3</sub> (1.2M or 8%)</li>
</ul>
</li>
<li>Vortex for 1 minute.</li>
<li>Centrifuge at room temperature 13000g for 2 minutes.</li>
<li>Pipette 600μl into 5mm NMR tube (take care not to disturb and particulate pellet at the bottom of the eppendorf).</li>
</ul>
<blockquote>
<p>Do not use TSP as a reference standard in plasma, serum or any other samples with a high protein content, since the compound binds to proteins resulting in a much reduced signal with a very broad line width.</p>
</blockquote></section><h1 class='nav-section-title-end'>Ended: Blood</h1>
                        <h2 class='nav-section-title' id='section-cells'>
                            Cells <a class='headerlink' href='#section-cells' title='Permanent link'>↵</a>
                        </h2>
                        <section class="print-page" id="sample%20preparation-cells"><h1 id="sample%20preparation-cells-intracellular-metabolite-extraction">Intracellular Metabolite Extraction</h1>
<p>Author: Marie Phelan
Revision Date: 4 October 2016</p>
<h2 id="sample%20preparation-cells-collection-from-cell-culture">Collection from Cell Culture</h2>
<p>Testing has concluded that cells cultured to 90% confluence in T25 flasks (~2 x 10<sup>5</sup> ) provide suitable level of metabolite extract if each flask represents one metabolomics sample.</p>
<blockquote>
<p>Levels of metabolites may be cell line dependent –please ask for details.</p>
</blockquote>
<ul>
<li>Optional (for adhering cells): Trypsinize (6 ml trypsin solution followed by 6 ml fresh media) cells at 80-90% confluence in T25 flasks into 15ml falcon tubes.</li>
<li>Harvest cells from each flask into separate pellets by centrifugation at 1000 g for 3 minutes at 10<span>&deg;</span>C.</li>
<li>Carefully pipette off the supernatant which can then be discarded.</li>
<li>Gently resuspend each cell pellet in 1 ml Phosphate Buffered Saline pH 7.4 (PBS).</li>
<li>Transfer cell suspensions into 1.5 ml Eppendorf.</li>
<li>Centrifuge Eppendorf at 1000 g for 5 minutes at 10<span>&deg;</span>C.</li>
<li>Carefully pipette off the supernatant which can then be discarded.</li>
<li>Freeze the cells rapidly in liquid nitrogen.</li>
<li>Store cells at -80<span>&deg;</span>.</li>
</ul>
<h2 id="sample%20preparation-cells-extraction-of-soluble-metabolites">Extraction of soluble metabolites</h2>
<ul>
<li>
<p>Prepare fresh ice-cold solvent solution: 50% HPLC grade acetonitrile, 50% double distilled water.</p>
</li>
<li>
<p>Add 500 μl ice-cold solvent solution into frozen Eppendorf containing pelleted cells (adjust volume for accurate weight).</p>
</li>
<li>
<p>Sonicate cells in solvent solution in Eppendorf placed in a beaker of ice using microtip (maximum frequency) in 3 x 30 second bursts to prevent heating.</p>
</li>
<li>
<p>Vortex the sample for 1 minute.</p>
</li>
<li>
<p>Centrifuge sample at 12000 g for 10 minutes at 4 <span>&deg;</span>C.</p>
</li>
<li>
<p>Collect the supernatant in clean 1.5 ml Eppendorf and lyophilise.</p>
</li>
<li>
<p>Store lyophilised material at -80 <span>&deg;</span>C until use.</p>
</li>
</ul>
<h2 id="sample%20preparation-cells-immediately-prior-to-nmr-acquisition-at-298k-to-be-carried-out-by-nmr-operative">Immediately prior to NMR acquisition at 298K (to be carried out by NMR operative)</h2>
<ul>
<li>To lyophilised material add:<ul>
<li>178 μl 100% D2O</li>
<li>20 μl 1mM sodium phosphate pH 7.4 in 100% D2O</li>
<li>0.2 μl 100mM TSP in 100% D2O</li>
<li>0.2 μl 1.2mM NaN3in 100% D2O</li>
</ul>
</li>
<li>Vortex for 1 minute.</li>
<li>Centrifuge at room temperature 12000 g for 2 minutes.</li>
<li>Pipette 190 μl into 3mm NMR tube (take care not to disturb and particulate pellet at the bottom of the eppendorf).</li>
</ul></section><h1 class='nav-section-title-end'>Ended: Cells</h1>
                        <h2 class='nav-section-title' id='section-cells-intracellular'>
                            Cells Intracellular <a class='headerlink' href='#section-cells-intracellular' title='Permanent link'>↵</a>
                        </h2>
                        <section class="print-page" id="sample%20preparation-cells%20intracellular"><h1 id="sample%20preparation-cells%20intracellular-intracellular-metabolite-extraction-from-cell-culture">Intracellular Metabolite Extraction from Cell Culture</h1>
<p>Authors: Marie Phelan &amp; Elinor Chapman 
Revision Date: 31 August 2018</p>
<h2 id="sample%20preparation-cells%20intracellular-urine-collection">Urine collection</h2>
<ul>
<li>Centrifuge samples at maximum g (min 4000) for 5 min, 4&deg;C</li>
<li>Pipette sample supernatants into 500μl aliquots in labelled Eppendorf (side not lid)</li>
<li>Snap freeze in liquid nitrogen and store at -80&deg;C, Samples may be stored for up to 9 months (<a href="https://pubs.acs.org/doi/10.1021/ac070212f">Maher, A.D., Zirah, S.F., Holmes, E. &amp; Nicholson, J.K. Anal. Chem. 79, 5204–5211 (2007)</a>).</li>
</ul>
<h2 id="sample%20preparation-cells%20intracellular-additional-preparation-for-method-2-3-lyophilised">Additional Preparation for method 2 &amp; 3- lyophilised:</h2>
<ul>
<li>Pierce lids of frozen Eppendorf.</li>
<li>Lyophilise overnight</li>
<li>Store lyophilised material at -80 &deg;C until use.</li>
</ul>
<h2 id="sample%20preparation-cells%20intracellular-immediately-prior-to-nmr-acquisition-at-300k-to-be-carried-out-by-nmr-operative">Immediately prior to NMR acquisition at 300K (to be carried out by NMR operative)</h2>
<h3 id="sample%20preparation-cells%20intracellular-method-1-non-lyophilised">Method 1 – non lyophilised</h3>
<p>Thaw samples at room temperature for 20-30 minutes.
 - Centrifuge for 5 minutes at 6000 c/min.
 - To 330μl urine sample add
    - 330μl 1M sodium phosphate pH 7.4 in 20% <sup>2</sup>H<sub>2</sub>O
    - 1μl 100mM TSP in 100% <sup>2</sup>H<sub>2</sub>O
    - 2.5μl NaN<sub>3</sub> (1.2M or 8%)</p>
<h3 id="sample%20preparation-cells%20intracellular-method-2-soluble-metabolites">Method 2- soluble metabolites</h3>
<ul>
<li>To lyophilised material add<ul>
<li>330 μl 100% <sup>2</sup>H<sub>2</sub>O</li>
<li>330 μl 1M sodium phosphate pH 7.4 in 100% <sup>2</sup>H<sub>2</sub>O</li>
<li>1 μl 100mM TSP in 100% <sup>2</sup>H<sub>2</sub>O</li>
<li>2.5μl NaN<sub>3</sub>(1.2M or 8%)</li>
</ul>
</li>
</ul>
<h3 id="sample%20preparation-cells%20intracellular-method-3-lipids">Method 3- lipids</h3>
<ul>
<li>Add the following to lyophilised material</li>
<li>600 μl C<sup>2</sup>HCl<sub>3</sub></li>
</ul>
<h3 id="sample%20preparation-cells%20intracellular-all-methods">All methods</h3>
<ul>
<li>Vortex for 30s.</li>
<li>Centrifuge at 4&deg;C 13000g for 2 minutes.</li>
<li>Pipette 600ul into 5mm NMR tube (take care not to disturb and particulate pellet at the bottom of the Eppendorf). </li>
</ul></section><h1 class='nav-section-title-end'>Ended: Cells Intracellular</h1>
                        <h2 class='nav-section-title' id='section-tissues'>
                            Tissues <a class='headerlink' href='#section-tissues' title='Permanent link'>↵</a>
                        </h2>
                        <section class="print-page" id="sample%20preparation-tissues"><h1 id="sample%20preparation-tissues-tissue-sample-preparation-soluble-metabolites">Tissue Sample Preparation – Soluble Metabolites</h1>
<p>Author: Marie Phelan
Revision Date: 4 October 2016</p>
<p>Once tissue is excised it is important to keep on ice and do the initial steps as quickly as possible:</p>
<ul>
<li>
<p>Keeping tissue on ice, divide tissue into ~10mg sections (accurately weighed) and place into individual 2ml Eppendorf, this procedure should not take more than 30 minutes.</p>
</li>
<li>
<p>Freeze the tissue sample rapidly in liquid nitrogen after division to immediately stop any enzymatic or chemical reactions.</p>
</li>
<li>
<p>Store tissue at -80&deg;C.</p>
</li>
</ul>
<h2 id="sample%20preparation-tissues-extraction-of-soluble-metabolites">Extraction of soluble metabolites</h2>
<ul>
<li>
<p>Prepare fresh ice-cold solvent solution: 50% HPLC grade acetonitrile, 50% water.</p>
<blockquote>
<p>50:50 Acetonitrile:Water buffer will select for primarily hydrophilic metabolites</p>
</blockquote>
</li>
<li>
<p>Add 100μl of solvent solution per 10mg tissue (adjust volume for accurate weight).</p>
</li>
<li>
<p>Sonicate tissue using microtip (10KHz) in solvent solution in an ice bath in 3x 30 second bursts to prevent heating.</p>
</li>
<li>
<p>Vortex the sample for 1 minute.</p>
</li>
<li>
<p>Centrifuge sample at 12000g for 5 minutes at 4&deg;C.</p>
</li>
<li>
<p>Collect the supernatant and lyophilise .</p>
</li>
<li>
<p>Store lyophilised material at -80&deg;C until use.</p>
</li>
</ul>
<p>## Immediately prior to NMR acquisition</p>
<ul>
<li>
<p>Add the following to lyophilised material</p>
<ul>
<li>70μl 1mM sodium phosphate pH 7.4 in 100% D<sub>2</sub>O</li>
<li>630μl H<sub>2</sub>O</li>
<li>0.7μl 100mM TSP in 100% D<sub>2</sub>O</li>
</ul>
</li>
<li>
<p>Vortex for 1 minute.</p>
</li>
<li>
<p>Centrifuge at room temperature 12000g for 2 minutes.</p>
</li>
<li>
<p>Pipette 600μl into 5mm NMR tube (take care not to disturb and particulate pellet at the bottom of the Eppendorf).</p>
</li>
</ul>
<blockquote>
<p>The typical sample size is 20 mg (wet mass) although in theory as little as 5 mg can be used. Homogenisation can be achieved via multiple methods with varying success dependent on amount and properties of tissue under study so please discuss optimisation process with us in advance. </p>
</blockquote></section><h1 class='nav-section-title-end'>Ended: Tissues</h1>
                        <h2 class='nav-section-title' id='section-urine'>
                            Urine <a class='headerlink' href='#section-urine' title='Permanent link'>↵</a>
                        </h2>
                        <section class="print-page" id="sample%20preparation-urine"><h1 id="sample%20preparation-urine-urine-sample-preparation">Urine Sample Preparation</h1>
<p>Author: Marie Phelan 
Revision date: 4 October 2016</p>
<blockquote>
<p>Collection tubes, which contain additives such as citrate and other added stabilizers should be avoided, since they give additional signals in the NMR spectra.</p>
</blockquote>
<h2 id="sample%20preparation-urine-collection">Collection</h2>
<ul>
<li>Collect ~0.8ml into plastic tubes (no additives) kept on ice. </li>
<li>Add 0.02% (final volume) NaN<sub>3</sub>.</li>
<li>Transfer samples on ice (not frozen) immediately to the laboratory.</li>
</ul>
<h2 id="sample%20preparation-urine-at-the-lab">At the lab</h2>
<ul>
<li>Centrifuge samples at 1,600g for 5 min at 4<span>&deg;</span>C (this step may be omitted if urine known to contain no bacterial contaminants). </li>
<li>Store supernatant at –80<span>&deg;</span>C until analysed. Samples may be stored for up to 9 months (Maher, A.D., Zirah, S.F., Holmes, E. &amp; Nicholson, J.K. Anal. Chem. 79, 5204–5211 (2007)).</li>
</ul>
<h2 id="sample%20preparation-urine-immediately-prior-to-nmr-acquisition-at-300k">Immediately prior to NMR acquisition at 300K</h2>
<ul>
<li>Thaw samples at room temperature for 20-30 minutes.</li>
<li>Centrifuge for 5 minutes at 6000 c/min.</li>
<li>To 900μl urine sample add:</li>
<li>100μl 1M sodium phosphate pH 7.4 in 100% D<sub>2</sub>O </li>
<li>1μl 100mM TSP in 100% D<sub>2</sub>O </li>
<li>Vortex for 1 minute.</li>
<li>Centrifuge at room temperature 13000g for 2 minutes. </li>
<li>Pipette 600ul into 5mm NMR tube (take care not to disturb and particulate pellet at the bottom of the eppendorf).</li>
</ul></section><h1 class='nav-section-title-end'>Ended: Urine</h1><h1 class='nav-section-title-end'>Ended: Sample Preparation</h1></div>





                
              </article>
            </div>
          
          
        </div>
        
          <a href="#" class="md-top md-icon" data-md-component="top" hidden>
            <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
            Back to top
          </a>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    <script id="__config" type="application/json">{"base": "/NMRmetab/", "features": ["navigation.tabs", "navigation.tabs.sticky", "navigation.tracking", "navigation.indexes", "navigation.top"], "search": "../assets/javascripts/workers/search.5bf1dace.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.config.lang": "en", "search.config.pipeline": "trimmer, stopWordFilter", "search.config.separator": "[\\s\\-]+", "search.placeholder": "Search", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version.title": "Select version"}, "version": {"provider": "mike"}}</script>
    
    
      <script src="../assets/javascripts/bundle.078830c0.min.js"></script>
      
        <script src="../js/print-site.js"></script>
      
        <script src="../javascripts/config.js"></script>
      
        <script src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>
      
        <script src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js"></script>
      
        <script src="https://unpkg.com/mermaid/dist/mermaid.min.js"></script>
      
    
  </body>
</html>