# Background Reading

Author: Marie Phelan
Revision Date: 1 March 2019

```mermaid
graph TB

subgraph <br>
AA((Study<br>Design)) --> B((Study<br>Design))
AA --> C((Sample<br>Collection))
AA --> D((Sample<br>Preparation))

AB((Data<br>Analysis)) --> E((Quality<br>Control))
AB --> F((Data<br>Formatting))
AB --> G((Common<br>Statistics))
AB --> H((Metabolite<br>Annotation<br>& Identities))

AC((Biological<br>Contextualisation)) --> I((Data<br>Deposition))
AC --> J((Link to<br>Biological<br>Pathway))
end

```

## Study Design

Metabolic profiling, metabolomic and metabonomic procedures for NMR spectroscopy of urine, plasma, serum and tissue extracts.
Beckonert O, Keun HC, Ebbels TM, Bundy J, Holmes E, Lindon JC, Nicholson JK.
Nat Protoc. 2007;2(11):2692-703.
https://www.ncbi.nlm.nih.gov/pubmed/18007604

NMR Metabolomics: A Comparison of the Suitability of Various Commonly used National Health Service Blood Collection Tubes
MM Phelan, LY Lian.
Current Metabolomics Volume 4 , Issue 1 , 2016 78 - 81
http://www.eurekaselect.com/135553

Using an NMR metabolomics approach to investigate the pathogenicity of amyloid-beta and alpha-synuclein
M. M. Phelan, E. Caamaño-Gutiérrez, M. S. Gant, R. X. Grosman, and J. Madine
Metabolomics. 2017; 13(12): 151.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5661010/

Transcriptomic and metabolomic data integration
Rachel Cavill Danyel Jennen Jos Kleinjans Jacob Jan Briedé
Briefings in Bioinformatics, Volume 17, Issue 5, 1 September 2016, Pages 891-901
https://academic.oup.com/bib/article/17/5/891/2262240

High-throughput serum NMR metabonomics for cost-effective holistic studies on systemic metabolism
Soininen P, Kangas AJ, Würtz P, Tukiainen T, Tynkkynen T, Laatikainen R, Järvelin MR, Kähönen M, Lehtimäki T, Viikari J, Raitakari OT, Savolainen MJ, Ala-Korpela M.
Analyst. 2009 Sep;134(9):1781-5. doi: 10.1039/b910205a. Epub 2009 Jul 30.
https://www.ncbi.nlm.nih.gov/pubmed/19684899

Databases and Software for NMR-Based Metabolomics
Ellinger JJ, Chylla RA, Ulrich EL, Markley JL.
Curr Metabolomics. 2013;1(1). doi: 10.2174/2213235X11301010028
https://www.ncbi.nlm.nih.gov/pubmed/24260723

Current practice of liquid chromatography-mass spectrometry in metabolomics and metabonomics
Gika HG, Theodoridis GA, Plumb RS, Wilson ID
J Pharm Biomed Anal. 2014 Jan;87:12-25. doi: 10.1016/j.jpba.2013.06.032. Epub 2013 Jul 17
https://www.ncbi.nlm.nih.gov/pubmed/23916607

Global metabolic profiling procedures for urine using UPLC-MS. Want EJ, Wilson ID, Gika H, Theodoridis G, Plumb RS, Shockcor J, Holmes E, Nicholson JK. Nat Protoc. 2010 Jun;5(6):1005-18. doi: 10.1038/nprot.2010.50. https://www.ncbi.nlm.nih.gov/pubmed/20448546

Proposed minimum reporting standards for chemical analysis Chemical Analysis Working Group (CAWG) Metabolomics Standards Initiative (MSI) Lloyd W. Sumner, Alexander Amberg, Dave Barrett, Michael H. Beale, Richard Beger, Clare A. Daykin, Teresa W.-M. Fan, Oliver Fiehn, Royston Goodacre, Julian L. Griffin, Thomas Hankemeier, Nigel Hardy, James Harnly, Richard Higashi, Joachim Kopka, Andrew N. Lane, John C. Lindon, Philip Marriott, Andrew W. Nicholls, Michael D. Reily, John J. Thaden, and Mark R. Viant Metabolomics. 2007 Sep; 3(3): 211\u2013221. https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3772505/

The role of reporting standards for metabolite annotation and identification in metabolomic studies. Salek RM, Steinbeck C, Viant MR, Goodacre R, Dunn WB. Gigascience. 2013 Oct 16;2(1):13. doi: 10.1186/2047-217X-2-13. https://www.ncbi.nlm.nih.gov/pubmed/24131531

Quality assurance and quality control processes: Summary of a metabolomics community questionnaire Warwick Dunn, David Broadhurst, Art Edison, Mark Viant, Daniel W. Bearden, Richard Beger, Claude Guillou https://www.nist.gov/publications/quality-assurance-and-quality-control-processes-summary-metabolomics-community

Data Analysis Proposed minimum reporting standards for chemical analysis Chemical Analysis Working Group (CAWG) Metabolomics Standards Initiative (MSI)
Lloyd W. Sumner,corresponding author Alexander Amberg, Dave Barrett, Michael H. Beale, Richard Beger, Clare A. Daykin, Teresa W.-M. Fan, Oliver Fiehn, Royston Goodacre, Julian L. Griffin, Thomas Hankemeier, Nigel Hardy, James Harnly, Richard Higashi, Joachim Kopka, Andrew N. Lane, John C. Lindon, Philip Marriott, Andrew W. Nicholls, Michael D. Reily, John J. Thaden, and Mark R. Viant
Metabolomics. 2007 Sep; 3(3): 211-221
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3772505/

The role of reporting standards for metabolite annotation and identification in metabolomic studies
Salek RM, Steinbeck C, Viant MR, Goodacre R, Dunn WB1
Gigascience. 2013 Oct 16;2(1):13. doi: 10.1186/2047-217X-2-13
https://www.ncbi.nlm.nih.gov/pubmed/24131531

Quality assurance and quality control processes: Summary of a metabolomics community questionnaire
Warwick Dunn, David Broadhurst, Art Edison, Mark Viant, Daniel W. Bearden, Richard Beger, Claude Guillou
https://www.nist.gov/publications/quality-assurance-and-quality-control-processes-summary-metabolomics-community

Global metabolic profiling procedures for urine using UPLC-MS
Want EJ, Wilson ID, Gika H, Theodoridis G, Plumb RS, Shockcor J, Holmes E, Nicholson JK
Nat Protoc. 2010 Jun;5(6):1005-18. doi: 10.1038/nprot.2010.50
https://www.ncbi.nlm.nih.gov/pubmed/20448546

State-of-the art data normalization methods improve NMR-based metabolomic analysis
Kohl, S. M., M. S. Klein, J. Hochrein, P. J. Oefner, R. Spang, W. Gronwald (2012)
Metabolomics 8, 146-160
https://www.ncbi.nlm.nih.gov/pubmed/22593726

Comparing groups for statistical differences: how to choose the right statistical test?
M Marusteri, V Bacarea
Biochemia Medica 2010;20(1):15-32
http://www.biochemia-medica.com/content/comparing-groups-statistical-differences-how-choose-right-statistical-test

Web-based inference of biological patterns, functions and pathways from metabolomic data using MetaboAnalyst
Jianguo Xia & David S Wishart
Nature Protocols 6, 743-760 (2011) doi:10.1038/nprot.2011.319
https://www.nature.com/articles/nprot.2011.319

Multivariate Analysis in Metabolomics
Bradley Worley & Robert Powers
Curr Metabolomics. 2013; 1(1): 92-107. doi: 10.2174/2213235X11301010092
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4465187/

## Biological Contextualisation

High-throughput serum NMR metabonomics for cost-effective holistic studies on systemic metabolism
Soininen P, Kangas AJ, Würtz P, Tukiainen T, Tynkkynen T, Laatikainen R, Järvelin MR, Kähönen M, Lehtimäki T, Viikari J, Raitakari OT, Savolainen MJ, Ala-Korpela M.Soininen
Analyst. 2009 Sep;134(9):1781-5 Analyst. 2009 Sep;134(9):1781-5. doi: 10.1039/b910205a. Epub 2009 Jul 30
https://www.ncbi.nlm.nih.gov/pubmed/19684899

MetaboLights-- an open-access general-purpose repository for metabolomics studies and associated meta-data
Haug et al Nucl. Acids Res. (2013) doi: 10.1093/nar/gks1004
https://academic.oup.com/nar/article/41/D1/D781/1050654

Databases and Software for NMR-Based Metabolomics
Ellinger JJ1, Chylla RA, Ulrich EL, Markley JL.
Curr Metabolomics. 2013;1(1). doi: 10.2174/2213235X11301010028
https://www.ncbi.nlm.nih.gov/pubmed/24260723

The CCPN Metabolomics Project: a fast protocol for metabolite identification by 2D-NMR
Chignola F1, Mari S, Stevens TJ, Fogh RH, Mannella V, Boucher W, Musco G.
Bioinformatics. 2011 Mar 15;27(6):885-6. doi: 10.1093/bioinformatics/btr013. Epub 2011 Jan 6.
https://www.ncbi.nlm.nih.gov/pubmed/21216771

MSEA: a web-based tool to identify biologically meaningful patterns in quantitative metabolomic data
Jianguo Xia David S. Wishart
Nucleic Acids Research, Volume 38, Issue suppl_2, 1 July 2010, Pages W71–W77
https://doi.org/10.1093/nar/gkq329 https://academic.oup.com/nar/article/38/suppl_2/W71/1101310

NMR Spectroscopy for Metabolomics and Metabolic Profiling
Cynthia K. Larive, Gregory A. Barding Jr. and Meredith M. Dinges
Anal. Chem., 2015, 87 (1), pp 133–146 DOI: 10.1021/ac504075g
https://pubs.acs.org/doi/abs/10.1021/ac504075g

HMDB: HMDB: the Human Metabolome Database
Wishart DS, et al
Nucleic Acids Res. 2007 35(Database issue): D521–D526 DOI: 10.1093/nar/gkl923
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1899095/

HMDB: a knowledgebase for the human metabolome
Wishart DS, et al
Nucleic Acids Res. 2009;37(Database issue):D603-10 DOI: 10.1093/nar/gkn810
https://www.ncbi.nlm.nih.gov/pubmed/18953024

HMDB 3.0 – The Human Metabolome Database in 2013
Wishart DS, et al
Nucleic Acids Res. 2013;41(Database issue):D801-7 DOI: 10.1093/nar/gks1065
https://www.ncbi.nlm.nih.gov/pubmed/23161693

HMDB 4.0: the human metabolome database for 2018
Wishart DS et al
Nucleic Acids Res. 2018 Jan 4;46(D1):D608-D617. doi: 10.1093/nar/gkx1089.
https://www.ncbi.nlm.nih.gov/pubmed/29140435

MetaboAnalyst: Web-based inference of biological patterns, functions and pathways from metabolomic data using MetaboAnalyst
Jianguo Xia & David S Wishart
Nature Protocols 6, 743–760 (2011) doi:10.1038/nprot.2011.319
https://www.nature.com/articles/nprot.2011.319

Using MetaboAnalyst 3.0 for Comprehensive Metabolomics Data Analysis.
Xia J, Wishart DS
Current Protocols in Bioinformatics 2016 55, p141
https://www.ncbi.nlm.nih.gov/pubmed/27603023

KEGG: Development of a chemical structure comparison method for integrated analysis of chemical and genomic information in the metabolic pathways (2003)
Hattori, M. et al
J. Am. Chem. Soc., 2003, 125 (39), pp 11853–11865 DOI:10.1021/ja036030u
https://pubs.acs.org/doi/abs/10.1021/ja036030u

SIMCOMP/SUBCOMP: chemical structure search servers for network analyses
Hattori, M. et al
Nucleic Acids Research, 38, Issue suppl_2, 2010, W652–W656 DOI:10.1093/nar/gkq367
https://academic.oup.com/nar/article/38/suppl_2/W652/1103855



