# Pathway Analysis in MetaboAnalyst

Authors: Marie Phelan & Rudi Grosman
Revision Date:2 nd November 2017

Go to MetaboAnalyst website (www.metaboanalyst.ca).

Click on ‘click here to start’, this will take you to the analysis selection page.

In the analysis selection page select ‘Pathway Analysis’.

In the input window enter the metabolites of interest.Makes sure you select the input type (available options are compound name, KEGG ID and HMDB ID).

If using compound names as input there is a name matching function that checks the names against the possible matches in the databse. In these cases the entry will be highlighted.

A yellow highlighted entry indicates an issue - click ‘View’ which will pop-up a window for possible entries. If an entry is highlighted with red, it means there is no match in the database.

Tick the correct compound from the list (select ‘None of the above’ if it is not listed). Then click ‘OK’.

Once the correct input selected MetaboAnalyst will fill the empty cells.

Once the input list has been checked (and corrected if necessary) click ‘submit’.

Select the organism that you want to perform pathway analysis on. The square brackets represent number of pathways available for the organisms. When the organism is selected scroll down.

Select **Over Representation Analysis** type and the **Pathway Topology Analysis** type (details can be found in the MetaboAnalyst tutorials). Select the compounds to be used and click ‘Submit’.

Each point on the **Pathway Overview** graph represents a pathway and when clicked on it will show the pathway in the next window. Red highlighted codes are the input metabolites. This section is followed by the list of pathways and their scores.

To download the results, click ‘Submit’ which will redirect to the download page.

On the download page either download individual sections or the whole analysis in a .zip file. To download everything select ‘Download.zip’.

## References

1. Xia, J., Sinelnikov, I., Han, B., and Wishart, D.S. (2015) [MetaboAnalyst 3.0 - making metabolomics more meaningful](http://doi.org/10.1093/nar/gkv380) . Nucl. Acids Res. 43, W251-257. 
2. Xia, J. and Wishart, D.S. (2011) [Web-based inference of biological patterns, functions and pathways from metabolomic data using MetaboAnalyst](http://doi.org/10.1038/nprot.2011.319) Nature Protocols 6 (6), 743-760. 
3. Xia, J. and Wishart, D.S. (2011) [Metabolomic data processing, analysis, and interpretation using MetaboAnalyst](http://doi.org/10.1002/0471250953.bi1410s34) Current Protocols in Bioinformatics, 14.10.1-14.10.48.
4. Xia, J., Broadhurst, D., Wilson, M. and Wishart, D. (2013) [Translational biomarker discovery in clinical metabolomics: an introductory tutorial](http://doi.org/10.1007/s11306-012-0482-9). Metabolomics, 9, 280-299. (biomarker analysis)
5. Xia, J. and Wishart, D.S. (2010) [MetPA: a web-based metabolomics tool for pathway analysis and visualization](http://doi.org/10.1093/bioinformatics/btq418). Bioinformatics, 26, 2342-2344 (metabolic pathway analysis)
6. Xia, J. and Wishart, D.S. (2010) [MSEA: a web-based tool to identify biologically meaningful patterns in quantitative metabolomic data](http://doi.org/10.1093/nar/gkq329). Nucleic Acids Research, 38, W71-77 (metabolite set enrichment analysis)

For MetaboAnalyst tutorials visit: http://www.metaboanalyst.ca/faces/docs/Tutorial.xhtml

## Alternative Software
MetExplore – tutorials available online

## Alternative Pathway Resources
HMDB1-3 Human metabolome database www.hmdb.ca SMPDB4-5 Small molecule pathways smpdb.ca KEGG6-10 Metabolites, genes and proteins pathway database www.kegg.jp 

1. Wishart DS, et al HMDB: the Human Metabolome Database (2007) DOI: [10.1093/nar/gkl923](https://doi.org/10.1093/nar/gkl923)
2. Wishart DS, et al HMDB: a knowledgebase for the human metabolome (2009) DOI: [10.1093/nar/gkn810](https://doi.org/10.1093/nar/gkn810)
3. Wishart DS, et al HMDB 3.0 – The Human Metabolome Database in 2013 (2013) DOI: [10.1093/nar/gks1065](https://doi.org/10.1093/nar/gks1065)
4. Wishart DS, et al SMPDB: The Small Molecule Pathway Database DOI: [10.1093/nar/gkp1002](https://doi.org/10.1093/nar/gkp1002)
5. Jewison T, et al SMPDB 2.0: Big Improvements to the Small Molecule Pathway Database DOI: [10.1093/nar/gkt1067](https://doi.org/10.1093/nar/gkt1067) 
6. Hashimoto, K.et al Comprehensive analysis of glycosyltransferases in eukaryotic genomes for structural and functional characterization of glycans (2009). DOI:[10.1016/j.carres.2009.03.001](https://doi.org/10.1016/j.carres.2009.03.001)
7. Hashimoto, K. et al The repertoire of desaturases and elongases reveals fatty acid variations in 56 eukaryotic genomes (2008). DOI:[10.1194/jlr.M700377-JLR200](https://doi.org/10.1194/jlr.M700377-JLR200)
8. Minowa, Y. et al Comprehensive analysis of distinctive polyketide and nonribosomal peptide structural motifs encoded in microbial genomes (2007). DOI:[10.1016/j.jmb.2007.02.099](https://doi.org/10.1016/j.jmb.2007.02.099)
9. Hattori, M. et al Development of a chemical structure comparison method for integrated analysis of chemical and genomic information in the metabolic pathways (2003). DOI:[10.1021/ja036030u](https://doi.org/10.1021/ja036030u)
10. Hattori, M. et al SIMCOMP/SUBCOMP: chemical structure search servers for network analyses (2010). DOI:[10.1093/nar/gkq367](https://doi.org/10.1093/nar/gkq367)